<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 6 PhD Research topic based | ReadingNotes: Have you read today’s papers?</title>
  <meta name="description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  <meta name="generator" content="bookdown 0.40 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 6 PhD Research topic based | ReadingNotes: Have you read today’s papers?" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 6 PhD Research topic based | ReadingNotes: Have you read today’s papers?" />
  
  <meta name="twitter:description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  

<meta name="author" content="Rongting Huang rthuang@connect.hku.hk" />


<meta name="date" content="2024-07-21" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="statmethods.html"/>
<link rel="next" href="proteomics.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Reading Notes-Rongting</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="" data-path="about-the-author.html"><a href="about-the-author.html"><i class="fa fa-check"></i>About the author</a></li>
<li class="chapter" data-level="" data-path="research-topics.html"><a href="research-topics.html"><i class="fa fa-check"></i>Research topics</a></li>
<li class="chapter" data-level="" data-path="about-the-book.html"><a href="about-the-book.html"><i class="fa fa-check"></i>About the book</a></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="sgcell.html"><a href="sgcell.html"><i class="fa fa-check"></i><b>2</b> SingleCell</a>
<ul>
<li class="chapter" data-level="2.1" data-path="sgcell.html"><a href="sgcell.html#technology"><i class="fa fa-check"></i><b>2.1</b> Technology</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="sgcell.html"><a href="sgcell.html#single-cell"><i class="fa fa-check"></i><b>2.1.1</b> single cell</a></li>
<li class="chapter" data-level="2.1.2" data-path="sgcell.html"><a href="sgcell.html#spatial-single-cell"><i class="fa fa-check"></i><b>2.1.2</b> spatial single cell</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="sgcell.html"><a href="sgcell.html#review"><i class="fa fa-check"></i><b>2.2</b> Review</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="sgcell.html"><a href="sgcell.html#review-2021-06"><i class="fa fa-check"></i><b>2.2.1</b> Review-2021-06</a></li>
<li class="chapter" data-level="2.2.2" data-path="sgcell.html"><a href="sgcell.html#review-2021-07"><i class="fa fa-check"></i><b>2.2.2</b> Review-2021-07</a></li>
<li class="chapter" data-level="2.2.3" data-path="sgcell.html"><a href="sgcell.html#review-2021-10"><i class="fa fa-check"></i><b>2.2.3</b> Review-2021-10</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="sgcell.html"><a href="sgcell.html#data-processing"><i class="fa fa-check"></i><b>2.3</b> Data processing</a></li>
<li class="chapter" data-level="2.4" data-path="sgcell.html"><a href="sgcell.html#clustering-methods"><i class="fa fa-check"></i><b>2.4</b> Clustering methods</a></li>
<li class="chapter" data-level="2.5" data-path="sgcell.html"><a href="sgcell.html#benchmarking"><i class="fa fa-check"></i><b>2.5</b> Benchmarking</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="technology-related.html"><a href="technology-related.html"><i class="fa fa-check"></i><b>3</b> Technology Related</a>
<ul>
<li class="chapter" data-level="3.1" data-path="technology-related.html"><a href="technology-related.html#scrna-seq-modelling"><i class="fa fa-check"></i><b>3.1</b> scRNA-seq modelling</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="technology-related.html"><a href="technology-related.html#nb-models-for-scrna-seq"><i class="fa fa-check"></i><b>3.1.1</b> NB models for scRNA-seq</a></li>
<li class="chapter" data-level="3.1.2" data-path="technology-related.html"><a href="technology-related.html#batch-effect-correction-methods-for-scrna-seq"><i class="fa fa-check"></i><b>3.1.2</b> Batch-effect correction methods for scRNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="technology-related.html"><a href="technology-related.html#hi-seq"><i class="fa fa-check"></i><b>3.2</b> Hi-Seq</a></li>
<li class="chapter" data-level="3.3" data-path="technology-related.html"><a href="technology-related.html#languages-and-compilers"><i class="fa fa-check"></i><b>3.3</b> languages and compilers</a></li>
<li class="chapter" data-level="3.4" data-path="technology-related.html"><a href="technology-related.html#singlecell-analysis-tools"><i class="fa fa-check"></i><b>3.4</b> singlecell analysis tools</a></li>
<li class="chapter" data-level="3.5" data-path="technology-related.html"><a href="technology-related.html#singlecell-analysis-sources"><i class="fa fa-check"></i><b>3.5</b> singlecell analysis sources</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="spatial-technology.html"><a href="spatial-technology.html"><i class="fa fa-check"></i><b>4</b> Spatial Technology</a>
<ul>
<li class="chapter" data-level="4.1" data-path="spatial-technology.html"><a href="spatial-technology.html#sequencing-based-st"><i class="fa fa-check"></i><b>4.1</b> Sequencing based ST</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="statmethods.html"><a href="statmethods.html"><i class="fa fa-check"></i><b>5</b> StatMethods</a>
<ul>
<li class="chapter" data-level="5.1" data-path="statmethods.html"><a href="statmethods.html#hmm-based-methods"><i class="fa fa-check"></i><b>5.1</b> HMM based methods</a></li>
<li class="chapter" data-level="5.2" data-path="statmethods.html"><a href="statmethods.html#em-based-methods"><i class="fa fa-check"></i><b>5.2</b> EM based methods</a></li>
<li class="chapter" data-level="5.3" data-path="statmethods.html"><a href="statmethods.html#vb-based-methods"><i class="fa fa-check"></i><b>5.3</b> VB based methods</a></li>
<li class="chapter" data-level="5.4" data-path="statmethods.html"><a href="statmethods.html#glm"><i class="fa fa-check"></i><b>5.4</b> GLM</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html"><i class="fa fa-check"></i><b>6</b> PhD Research topic based</a>
<ul>
<li class="chapter" data-level="6.1" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#cnv-calling"><i class="fa fa-check"></i><b>6.1</b> CNV calling</a>
<ul>
<li class="chapter" data-level="6.1.1" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#breaking-point-detection"><i class="fa fa-check"></i><b>6.1.1</b> breaking point detection</a></li>
<li class="chapter" data-level="6.1.2" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#related-research"><i class="fa fa-check"></i><b>6.1.2</b> Related research</a></li>
<li class="chapter" data-level="6.1.3" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#smoothing-strategies-in-rdr"><i class="fa fa-check"></i><b>6.1.3</b> Smoothing strategies in RDR</a></li>
<li class="chapter" data-level="6.1.4" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#pubmon"><i class="fa fa-check"></i><b>6.1.4</b> PUBMON</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#clonal-tree"><i class="fa fa-check"></i><b>6.2</b> Clonal Tree</a></li>
<li class="chapter" data-level="6.3" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#deconvolution"><i class="fa fa-check"></i><b>6.3</b> Deconvolution</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#deconvolution-of-bulk-tissue-and-spatial-transcriptomic-data"><i class="fa fa-check"></i><b>6.3.1</b> Deconvolution of bulk tissue and spatial transcriptomic data</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#omics-integration"><i class="fa fa-check"></i><b>6.4</b> Omics integration</a></li>
<li class="chapter" data-level="6.5" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#spatial-transcriptomics"><i class="fa fa-check"></i><b>6.5</b> Spatial transcriptomics</a></li>
<li class="chapter" data-level="6.6" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#spatial-transcriptomics-methods"><i class="fa fa-check"></i><b>6.6</b> Spatial transcriptomics Methods</a></li>
<li class="chapter" data-level="6.7" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#spatial-datasets-collection"><i class="fa fa-check"></i><b>6.7</b> Spatial Datasets Collection</a></li>
<li class="chapter" data-level="6.8" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#d-biology"><i class="fa fa-check"></i><b>6.8</b> 3D biology</a></li>
<li class="chapter" data-level="6.9" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#sequencing-data-relative-knowledge"><i class="fa fa-check"></i><b>6.9</b> Sequencing data relative Knowledge</a></li>
<li class="chapter" data-level="6.10" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#image-data-relative-knowledge"><i class="fa fa-check"></i><b>6.10</b> Image data relative Knowledge</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="proteomics.html"><a href="proteomics.html"><i class="fa fa-check"></i><b>7</b> Proteomics</a>
<ul>
<li class="chapter" data-level="7.1" data-path="proteomics.html"><a href="proteomics.html#ptm-cross-talk"><i class="fa fa-check"></i><b>7.1</b> PTM cross-talk</a></li>
<li class="chapter" data-level="7.2" data-path="proteomics.html"><a href="proteomics.html#ptm-cluster"><i class="fa fa-check"></i><b>7.2</b> PTM Cluster</a></li>
<li class="chapter" data-level="7.3" data-path="proteomics.html"><a href="proteomics.html#ptm-and-cancer"><i class="fa fa-check"></i><b>7.3</b> PTM and cancer</a></li>
<li class="chapter" data-level="7.4" data-path="proteomics.html"><a href="proteomics.html#protein-structure"><i class="fa fa-check"></i><b>7.4</b> Protein structure</a>
<ul>
<li class="chapter" data-level="7.4.1" data-path="proteomics.html"><a href="proteomics.html#protein-folding-prediction-background"><i class="fa fa-check"></i><b>7.4.1</b> Protein Folding Prediction Background</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="epigenomics.html"><a href="epigenomics.html"><i class="fa fa-check"></i><b>8</b> Epigenomics</a></li>
<li class="chapter" data-level="9" data-path="pubmon-1.html"><a href="pubmon-1.html"><i class="fa fa-check"></i><b>9</b> Pubmon</a>
<ul>
<li class="chapter" data-level="9.1" data-path="pubmon-1.html"><a href="pubmon-1.html#phd-year-1"><i class="fa fa-check"></i><b>9.1</b> 2020-2021 PhD year 1</a></li>
<li class="chapter" data-level="9.2" data-path="pubmon-1.html"><a href="pubmon-1.html#phd-year-2"><i class="fa fa-check"></i><b>9.2</b> 2021-2022 PhD year 2</a>
<ul>
<li class="chapter" data-level="9.2.1" data-path="pubmon-1.html"><a href="pubmon-1.html#resources"><i class="fa fa-check"></i><b>9.2.1</b> Resources</a></li>
<li class="chapter" data-level="9.2.2" data-path="pubmon-1.html"><a href="pubmon-1.html#ppi-topic"><i class="fa fa-check"></i><b>9.2.2</b> PPI topic</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="pubmon-1.html"><a href="pubmon-1.html#phd-year-4"><i class="fa fa-check"></i><b>9.3</b> 2023-2024 PhD year 4</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="method-of-the-year.html"><a href="method-of-the-year.html"><i class="fa fa-check"></i><b>10</b> Method of the Year</a>
<ul>
<li class="chapter" data-level="10.1" data-path="method-of-the-year.html"><a href="method-of-the-year.html#method-of-the-year-2022"><i class="fa fa-check"></i><b>10.1</b> Method of the Year 2022:</a></li>
<li class="chapter" data-level="10.2" data-path="method-of-the-year.html"><a href="method-of-the-year.html#method-of-the-year-2021-protein-structure-prediction"><i class="fa fa-check"></i><b>10.2</b> Method of the Year 2021: Protein structure prediction</a></li>
<li class="chapter" data-level="10.3" data-path="method-of-the-year.html"><a href="method-of-the-year.html#method-of-the-year-2020-spatially-resolved-transcriptomics"><i class="fa fa-check"></i><b>10.3</b> Method of the Year 2020: spatially resolved transcriptomics</a></li>
<li class="chapter" data-level="10.4" data-path="method-of-the-year.html"><a href="method-of-the-year.html#method-of-the-year-2019-single-cell-multimodal-omics"><i class="fa fa-check"></i><b>10.4</b> Method of the Year 2019: Single-cell multimodal omics</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">ReadingNotes: Have you read today’s papers?</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="phd-research-topic-based" class="section level1 hasAnchor" number="6">
<h1><span class="header-section-number">Chapter 6</span> PhD Research topic based<a href="phd-research-topic-based.html#phd-research-topic-based" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="cnv-calling" class="section level2 hasAnchor" number="6.1">
<h2><span class="header-section-number">6.1</span> CNV calling<a href="phd-research-topic-based.html#cnv-calling" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="breaking-point-detection" class="section level3 hasAnchor" number="6.1.1">
<h3><span class="header-section-number">6.1.1</span> breaking point detection<a href="phd-research-topic-based.html#breaking-point-detection" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li>4 CNV breakpoint detection methods (2021-07-17 Group meeting)</li>
</ul>
<ol style="list-style-type: decimal">
<li>CHISEL: <a href="https://www.nature.com/articles/s41587-020-0661-6#Sec8" class="uri">https://www.nature.com/articles/s41587-020-0661-6#Sec8</a> (see global clustering subsection)</li>
</ol>
<ul>
<li>seemingly no breakpoint detection, but rather a global clustering (ie. entry-wise for a bin-by-cell matrix), thus the resolution of CNV is the bin size (5MB)</li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>Alleloscope: <a href="https://www.nature.com/articles/s41587-021-00911-w#Sec10" class="uri">https://www.nature.com/articles/s41587-021-00911-w#Sec10</a> (see segmentation subsection)</li>
</ol>
<ul>
<li>HMM on a pooled cells (pseudo-bulk?) with pre-defined Gaussian means and variance for each state</li>
</ul>
<ol start="3" style="list-style-type: decimal">
<li>InferCNV: <a href="https://github.com/broadinstitute/inferCNV/wiki/inferCNV-HMM-based-CNV-Prediction-Methods" class="uri">https://github.com/broadinstitute/inferCNV/wiki/inferCNV-HMM-based-CNV-Prediction-Methods</a></li>
</ol>
<ul>
<li>i6-HMM generates in silico spike-in; seemingly define CNV region (segment) on cluster instead of cell, but using noise model on each cell (not quite sure from the doc).</li>
</ul>
<ol start="4" style="list-style-type: decimal">
<li>CopyKat: <a href="https://www.nature.com/articles/s41587-020-00795-2#Sec9" class="uri">https://www.nature.com/articles/s41587-020-00795-2#Sec9</a></li>
</ol>
<ul>
<li>KS test for whether to two neighbour bins should be joined, by using the posterior samples of Gamma-Poisson posterior. Seemingly using noise model on each cell within a cluster</li>
</ul>
<p>FACLON
<a href="https://academic.oup.com/nar/article/43/4/e23/2410993" class="uri">https://academic.oup.com/nar/article/43/4/e23/2410993</a></p>
<div class="figure">
<img src="figs/CNV/HATCHet.jpg" alt="" />
<p class="caption">Overview of HATCHet algorithm</p>
</div>
<p><a href="https://www.nature.com/articles/s41467-020-17967-y/figures/1" class="uri">https://www.nature.com/articles/s41467-020-17967-y/figures/1</a></p>
<p><strong>a</strong> HATCHet takes in input DNA sequencing data from multiple bulk tumor samples of the same patient and has five steps.
<strong>b</strong> First, HATCHet calculates the RDRs and BAFs in bins of the reference genome (black squares). Here, we show two tumor samples p and q.
<strong>c</strong> Second, HATCHet clusters the bins based on RDRs and BAFs globally along the entire genome and jointly across samples p and q. Each cluster (color) includes bins with the same copy-number state within each clone present in p or q.
<strong>d</strong> Third, HATCHet estimates two values for the fractional copy number of each cluster by scaling RDRs. If there is no WGD, the identification of the cluster (magenta) with copy-number state (1, 1) is sufficient and RDRs are scaled correspondingly.
If a WGD occurs, HATCHet identifies an additional cluster with identical copy-number state in all tumor clones. Dashed black horizontal lines in the scaled BAF-RDR plot represent values of fractional copy numbers that correspond to clonal CNAs.
<strong>e</strong> Fourth, HATCHet factors the allele-specific fractional copy numbers FA, FB into the allele-specific copy numbers A, B, respectively, and the clone proportions U. Here, there is a normal clone and 3 tumor clones.
<strong>f</strong> Last, HATCHet’s model-selection criterion identifies the matrices A, B, and U in the factorization while evaluating the fit according to both the inferred number of clones and presence/absence of a WGD.
<strong>g</strong> HATCHet outputs allele- and clone-specific copy numbers (with the color of the corresponding clone) and clone proportions (in the top right part of each plot) for each sample.
Clusters are classified according to the inference of unique/different copy-number states in each sample (sample-clonal/subclonal) and across all tumor clones (tumor-clonal/subclonal).</p>
<div class="figure">
<img src="figs/CNV/chisel.jpg" alt="" />
<p class="caption">Overview of chisel algorithm</p>
</div>
<p><a href="https://www.nature.com/articles/s41587-020-0661-6/figures/1" class="uri">https://www.nature.com/articles/s41587-020-0661-6/figures/1</a></p>
<p><strong>a</strong>, CHISEL computes RDRs and BAFs in low-coverage (&lt;0.05× per cell) single-cell DNA sequencing data (top left). Read counts from 2,000 individual cells (rows) in 5-Mb genomic bins (columns) across three chromosomes (gray rectangles in first row) are shown.
For each bin in each cell, CHISEL computes the RDR (top) by normalizing the observed read counts. CHISEL computes the BAF in each bin and cell (bottom) by first performing referenced-based phasing of germline SNPs in 50-kb haplotype blocks (magenta and green) and then phasing all these blocks jointly across all cells.
<strong>b</strong>, CHISEL clusters RDRs and BAFs globally along the genome and jointly across all cells resulting here in five clusters of genomic bins (red, blue, purple, yellow and gray) with distinct copy-number states.
<strong>c</strong>, CHISEL infers a pair {c^t,cˇt} of allele-specific copy numbers for each cluster by determining whether the allele-specific copy numbers of the largest balanced (BAF of ~0.5) cluster are equal to {1, 1} (diploid), {2, 2} (tetraploid) or are higher ploidy.
<strong>d</strong>, CHISEL infers haplotype-specific copy numbers (at, bt) by phasing the allele-specific copy numbers {c^t,cˇt} consistently across all cells.
<strong>e</strong>, CHISEL clusters tumor cells into clones according to their haplotype-specific copy numbers. Here, a diploid clone (light gray) and two tumor clones (red and blue) are obtained.
A phylogenetic tree describes the evolution of these clones. Somatic SNVs are derived from pseudo-bulk samples and placed on the branches of the tree.</p>
<div class="figure">
<img src="figs/CNV/TITAN.jpg" alt="" />
<p class="caption">Overview of TITAN local clustering for segemetation</p>
</div>
<div class="figure">
<img src="figs/CNV/inferCNV_i3HMM_model.png" alt="" />
<p class="caption">Overview of inferCNV i3HMM</p>
</div>
<div class="figure">
<img src="figs/CNV/infercnv_i6HMM_model.png" alt="" />
<p class="caption">Overview of inferCNV i6HMM</p>
</div>
<div class="figure">
<img src="figs/CNV/HoneyBADGER.png" alt="" />
<p class="caption">Overview of HoneyBADGER method</p>
</div>
<div class="figure">
<img src="figs/CNV/Alleloscope.jpg" alt="" />
<p class="caption">Overview of Alleloscope algorithm</p>
</div>
<div class="figure">
<img src="figs/CNV/copykat.jpg" alt="" />
<p class="caption">Overview of copykat algorithm</p>
</div>
</div>
<div id="related-research" class="section level3 hasAnchor" number="6.1.2">
<h3><span class="header-section-number">6.1.2</span> Related research<a href="phd-research-topic-based.html#related-research" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><strong>clonealign</strong>: statistical integration of independent single-cell RNA and DNA sequencing data from human cancers</p>
<p>However, independently sampled single-cell measurements introduce a new analytical challenge of how to associate cells across each modality. Assuming a population structure with a fixed number of clones, this can be expressed as a mapping problem, whereby cells measured with transcriptome assays must be aligned to those measured with a genome assay.</p>
<p><img src="figs/CNV/clonealign.jpg" alt="clonealign overview" />
In order to relate the independent measurements, we assume that an increase in the copy number of a gene will result in a corresponding increase in that gene’s expression and vice versa (Fig. 1b)
a relationship previously observed in joint RNA-DNA assays in bulk tissues [12] and at the single-cell level [9, 10, 13].</p>
<p><strong>CHISEL</strong></p>
<p><strong>inferCNV</strong></p>
<p><strong>copyKAT</strong></p>
<p><strong>CaSpER</strong></p>
<p><strong>HoneyBADGER</strong></p>
</div>
<div id="smoothing-strategies-in-rdr" class="section level3 hasAnchor" number="6.1.3">
<h3><span class="header-section-number">6.1.3</span> Smoothing strategies in RDR<a href="phd-research-topic-based.html#smoothing-strategies-in-rdr" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>-TODO</p>
</div>
<div id="pubmon" class="section level3 hasAnchor" number="6.1.4">
<h3><span class="header-section-number">6.1.4</span> PUBMON<a href="phd-research-topic-based.html#pubmon" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><a href="https://www.biorxiv.org/content/10.1101/2021.11.25.469995v1">Precise identification of cancer cells from allelic imbalances in single cell transcriptomes</a></p>
<p>this paper used BAF information to identify cancer cells, seems quite relevant.</p>
<p><strong>2022-03-09</strong></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2021.06.04.447031v1.full">Evolutionary tracking of cancer haplotypes at single-cell resolution</a></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2022.02.07.479314v1.full">Haplotype-enhanced inference of somatic copy number profiles from single-cell transcriptomes</a></p>
<ul>
<li><p><strong>Numbat</strong></p></li>
<li><p>introduction</p></li>
</ul>
<p>Existing approaches for CNV detection from scRNA-seq do not utilize the prior knowledge of haplotypes, or the individual-specific configuration of variant alleles on the two homologous chromosomes, which can enable more sensitive detection of allelic imbalance.</p>
<p>The utility of phasing in detecting CNV signals from scRNA-based assays, however, has not been explored.</p>
<p>We therefore developed a computational method, Numbat, which integrates expression, allele, and haplotype information derived from population-based phasing to comprehensively characterize the CNV landscape in single-cell transcriptomes.</p>
<p>Numbat does not require sample-matched DNA data or a priori genotyping, and is widely applicable to a wide range of experimental settings and cancer types.</p>
<ul>
<li>Results</li>
</ul>
<p><strong>Enhanced detection of subclonal allelic imbalances using population-based haplotype phasing</strong></p>
<p>Prior phasing information can effectively amplify weak allelic imbalance signals of individual SNPs induced by the CNV, by exposing joint behavior of entire haplotype sequences and thereby increasing the statistical power.</p>
<p>The ability to infer phasing between genes is particularly useful for CNV inference, as it provides means to overcome stochastic allele-specific expression effects which give rise to bursts of gene-specific allelic imbalances in individual cells.</p>
<p>The differential phasing accuracy from within and between genes reflects the fact that the strength of genetic linkage decays with increasing distance (Supplementary Figure 1a).</p>
<p>To reflect the decay in phasing strength over longer genetic distances, we introduced site-specific transition probabilities between haplotype states in the Numbat allele HMM (see Methods).</p>
<p><strong>Accurate copy number inference from single-cell transcriptomes</strong></p>
<p>To increase robustness, Numbat models gene expression as integer read counts using a discrete Poisson Lognormal mixture distribution, and accounts for excess variance in the allele frequency (e.g. due to allele-specific detection or transcriptional bursts) using a Beta-Binomial distribution.</p>
<p><strong>Iterative strategy to decompose tumor clonal architecture</strong></p>
<p><strong>Reliable identification of cancer cells in the tumor microenvironment</strong></p>
<p><strong>Allele-specific CNV analysis reveals additional subclonal complexity</strong></p>
<p><strong>Unraveling the interplay between genetic and transcriptional heterogeneity in tumor evolution</strong></p>
<ul>
<li><p>Discussion</p></li>
<li><p>Methods</p></li>
</ul>

</div>
</div>
<div id="clonal-tree" class="section level2 hasAnchor" number="6.2">
<h2><span class="header-section-number">6.2</span> Clonal Tree<a href="phd-research-topic-based.html#clonal-tree" class="anchor-section" aria-label="Anchor link to header"></a></h2>

</div>
<div id="deconvolution" class="section level2 hasAnchor" number="6.3">
<h2><span class="header-section-number">6.3</span> Deconvolution<a href="phd-research-topic-based.html#deconvolution" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="deconvolution-of-bulk-tissue-and-spatial-transcriptomic-data" class="section level3 hasAnchor" number="6.3.1">
<h3><span class="header-section-number">6.3.1</span> Deconvolution of bulk tissue and spatial transcriptomic data<a href="phd-research-topic-based.html#deconvolution-of-bulk-tissue-and-spatial-transcriptomic-data" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Related papers are as follows:</p>
<ul>
<li><a href="https://www.nature.com/articles/s41587-019-0114-2">CIBERSORTx</a></li>
<li><a href="https://www.nature.com/articles/s41587-021-00830-w">RCTD</a></li>
<li><a href="https://academic.oup.com/nar/article/49/9/e50/6129341">SPOTlight</a></li>
<li><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02362-7">SpatialDWLS</a></li>
</ul>
<p>From xiunan’s pre</p>
<p><a href="https://www.biorxiv.org/content/10.1101/2021.06.15.448381v1">Reference-free cell-type deconvolution of pixel-resolution spatially resolved transcriptomics data</a></p>
<p>Yuanhua add</p>

</div>
</div>
<div id="omics-integration" class="section level2 hasAnchor" number="6.4">
<h2><span class="header-section-number">6.4</span> Omics integration<a href="phd-research-topic-based.html#omics-integration" class="anchor-section" aria-label="Anchor link to header"></a></h2>

</div>
<div id="spatial-transcriptomics" class="section level2 hasAnchor" number="6.5">
<h2><span class="header-section-number">6.5</span> Spatial transcriptomics<a href="phd-research-topic-based.html#spatial-transcriptomics" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<ul>
<li>202109 Read in depth (lead by xianjie)</li>
</ul>
<p><strong><a href="https://www.biorxiv.org/content/10.1101/2021.07.12.452018v1.full">The spatial landscape of clonal somatic mutations in benign and malignant tissue</a><span class="citation">(<a href="#ref-erickson2021spatial" role="doc-biblioref">Erickson et al. 2021</a>)</span></strong></p>
<p>Keywords: CNV</p>
<p><strong><a href="https://www.sciencedirect.com/science/article/pii/S0092867420306723">Multimodal analysis of composition and spatial architecture in human squamous cell carcinoma</a><span class="citation">(<a href="#ref-zhang2021supergnova" role="doc-biblioref">Zhang et al. 2021</a>)</span></strong></p>
<p>Keywords: alignment of scRNA AND ST</p>
<ul>
<li>2022 spatial and CNV</li>
</ul>
<p><strong><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02653-7">Statistical and machine learning methods for spatially resolved transcriptomics data analysis</a><span class="citation">(<a href="#ref-zeng2022statistical" role="doc-biblioref">Zeng et al. 2022</a>)</span></strong></p>
<p><strong><a href="https://pubmed.ncbi.nlm.nih.gov/33022659/">STARCH: copy number and clone inference from spatial transcriptomics data</a><span class="citation">(<a href="#ref-elyanow2021starch" role="doc-biblioref">Elyanow et al. 2021</a>)</span></strong></p>
<p><strong><a href="https://www.biorxiv.org/content/10.1101/2021.07.12.452018v1.abstract">The spatial landscape of clonal somatic mutations in benign and malignant tissue</a><span class="citation">(<a href="#ref-erickson2021spatial" role="doc-biblioref">Erickson et al. 2021</a>)</span></strong></p>
<p>Keywords: spatial; CNV</p>

</div>
<div id="spatial-transcriptomics-methods" class="section level2 hasAnchor" number="6.6">
<h2><span class="header-section-number">6.6</span> Spatial transcriptomics Methods<a href="phd-research-topic-based.html#spatial-transcriptomics-methods" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<ul>
<li>Readinglist</li>
</ul>
<p><a href="https://www.nature.com/articles/s41592-024-02325-3">Systematic comparison of sequencing-based spatial transcriptomic methods</a></p>
<div class="figure">
<img src="figs/spatialtech/spatialmethodscomparasion.webp" alt="" />
<p class="caption">spatialmethodscomparasion</p>
</div>
</div>
<div id="spatial-datasets-collection" class="section level2 hasAnchor" number="6.7">
<h2><span class="header-section-number">6.7</span> Spatial Datasets Collection<a href="phd-research-topic-based.html#spatial-datasets-collection" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<ul>
<li>CNA focused exploration</li>
</ul>
<ol style="list-style-type: decimal">
<li>DMG+GBM Dataset</li>
</ol>
<p><a href="https://www.nature.com/articles/s41467-023-36707-6">Spatial transcriptomics reveals niche-specific enrichment and vulnerabilities of radial glial stem-like cells in malignant gliomas</a></p>
<p>Diffuse midline glioma-H3K27M mutant (DMG) and glioblastoma (GBM) are the most lethal brain tumors that primarily occur in pediatric and adult patients, respectively. Both tumors exhibit significant heterogeneity, shaped by distinct genetic/epigenetic drivers, transcriptional programs including RNA splicing, and microenvironmental cues in glioma niches. However, the spatial organization of cellular states and niche-specific regulatory programs remain to be investigated. Here, we perform a spatial profiling of DMG and GBM combining short- and long-read spatial transcriptomics, and single-cell transcriptomic datasets. We identify clinically relevant transcriptional programs, RNA isoform diversity, and multi-cellular ecosystems across different glioma niches. We find that while the tumor core enriches for oligodendrocyte precursor-like cells, radial glial stem-like (RG-like) cells are enriched in the neuron-rich invasive niche in both DMG and GBM. Further, we identify niche-specific regulatory programs for RG-like cells, and functionally confirm that FAM20C mediates invasive growth of RG-like cells in a neuron-rich microenvironment in a human neural stem cell derived orthotopic DMG model. Together, our results provide a blueprint for understanding the spatial architecture and niche-specific vulnerabilities of DMG and GBM<span class="citation">(<a href="#ref-ren2023spatial" role="doc-biblioref">Ren et al. 2023</a>)</span>.</p>
<p><a href="https://ngdc.cncb.ac.cn/gsa-human/browse/HRA001865">data request</a></p>
<div class="figure">
<img src="figs/spatialDatasets/DMG_GBM_dataset.webp" alt="" />
<p class="caption">multi-layered_GBM Graphical abstract</p>
</div>
<ol start="2" style="list-style-type: decimal">
<li>GBM_ST</li>
</ol>
<p><a href="https://www.sciencedirect.com/science/article/pii/S1535610822002203?ref=pdf_download&amp;fr=RR-2&amp;rr=82a4735d0c8f20e1">Spatially resolved multi-omics deciphers bidirectional tumor-host interdependence in glioblastoma</a></p>
<p><strong>Summary</strong>
Glioblastomas are malignant tumors of the central nervous system hallmarked by subclonal diversity and dynamic adaptation amid developmental hierarchies. The source of dynamic reorganization within the spatial context of these tumors remains elusive. Here, we characterized glioblastomas by spatially resolved transcriptomics, metabolomics, and proteomics. By deciphering regionally shared transcriptional programs across patients, we infer that glioblastoma is organized by spatial segregation of lineage states and adapts to inflammatory and/or metabolic stimuli, reminiscent of the reactive transformation in mature astrocytes. Integration of metabolic imaging and imaging mass cytometry uncovered locoregional tumor-host interdependence, resulting in spatially exclusive adaptive transcriptional programs. Inferring copy-number alterations emphasizes a spatially cohesive organization of subclones associated with reactive transcriptional programs, confirming that environmental stress gives rise to selection pressure. A model of glioblastoma stem cells implanted into human and rodent neocortical tissue mimicking various environments confirmed that transcriptional states originate from dynamic adaptation to various environments<span class="citation">(<a href="#ref-ravi2022spatially" role="doc-biblioref">Ravi et al. 2022</a>)</span>.</p>
<div class="figure">
<img src="figs/spatialDatasets/multi-omicsGBM.jpg" alt="" />
<p class="caption">multi-layered_GBM Graphical abstract</p>
</div>
<p><strong>Highlights</strong></p>
<ul>
<li><p>Five spatially distinct transcriptional programs are identified in glioblastomas</p></li>
<li><p>Hypoxia induces defined transcriptional and genomic responses, including CNAs</p></li>
<li><p>Immunosuppressive tumor-myeloid cell interactions are enhanced in segregated niches</p></li>
<li><p>Non-stress environments support subtype transition towards developmental stages</p></li>
</ul>
<ol start="3" style="list-style-type: decimal">
<li>GBM_Multilayer</li>
</ol>
<p><a href="https://www.cell.com/cell/fulltext/S0092-8674(24)00320-9?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867424003209%3Fshowall%3Dtrue">Integrative spatial analysis reveals a multi-layered organization of glioblastoma</a></p>
<p><strong>Abstract</strong></p>
<p>Glioma contains malignant cells in diverse states. Here, we combine spatial transcriptomics, spatial proteomics, and computational approaches to define glioma cellular states and uncover their organization. We find three prominent modes of organization. First, gliomas are composed of small local environments, each typically enriched with one major cellular state. Second, specific pairs of states preferentially reside in proximity across multiple scales. This pairing of states is consistent across tumors. Third, these pairwise interactions collectively define a global architecture composed of five layers. Hypoxia appears to drive the layers, as it is associated with a long-range organization that includes all cancer cell states. Accordingly, tumor regions distant from any hypoxic/necrotic foci and tumors that lack hypoxia such as low-grade IDH-mutant glioma are less organized. In summary, we provide a conceptual framework for the organization of cellular states in glioma, highlighting hypoxia as a long-range tissue organizer.<span class="citation">(<a href="#ref-greenwald2024integrative" role="doc-biblioref">Greenwald et al. 2024</a>)</span></p>
<p>Keywords: glioblastoma; glioma; hypoxia; intratumor heterogeneity; spatial proteomics; spatial transcriptomics.</p>
<div class="figure">
<img src="figs/spatialDatasets/multi-layered_GBM.jpg" alt="" />
<p class="caption">multi-layered_GBM Graphical abstract</p>
</div>
<ol start="4" style="list-style-type: decimal">
<li>Liver</li>
</ol>
<p><a href="https://www.science.org/doi/10.1126/sciadv.abg3750">Comprehensive analysis of spatial architecture in primary liver cancer</a></p>
<p>Heterogeneity is the major challenge for cancer prevention and therapy. Here, we first constructed high-resolution spatial transcriptomes of primary liver cancers (PLCs) containing 84,823 spots within 21 tissues from seven patients. The progressive comparison of spatial tumor microenvironment (TME) characteristics from nontumor to leading-edge to tumor regions revealed that the tumor capsule potentially affects intratumor spatial cluster continuity, transcriptome diversity, and immune cell infiltration. Locally, we found that the bidirectional ligand-receptor interactions at the 100-??m-wide cluster-cluster boundary contribute to maintaining intratumor architecture and the PROM1+ and CD47+ cancer stem cell niches are related to TME remodeling and tumor metastasis. Last, we proposed a TLS-50 signature to accurately locate tertiary lymphoid structures (TLSs) spatially and unveiled that the distinct composition of TLSs is shaped by their distance to tumor cells. Our study provides previous unknown insights into the diverse tumor ecosystem of PLCs and has potential benefits for cancer intervention.<span class="citation">(<a href="#ref-wu2021comprehensive" role="doc-biblioref">Wu et al. 2021</a>)</span></p>
<p><a href="https://ngdc.cncb.ac.cn/gsa-human/browse/HRA000437">data request</a></p>
<div class="figure">
<img src="figs/spatialDatasets/sciadv.abg3750-f1.jpg" alt="" />
<p class="caption">Liver cancer fig1</p>
</div>
<ol start="5" style="list-style-type: decimal">
<li>NPC</li>
</ol>
<p><a href="https://www.nature.com/articles/s41467-023-37614-6">Nasopharyngeal carcinoma cells promote regulatory T cell development and suppressive activity via CD70-CD27 interaction</a></p>
<p><strong>Abstract</strong>
Despite the intense CD8+ T-cell infiltration in the tumor microenvironment of nasopharyngeal carcinoma, anti-PD-1 immunotherapy shows an unsatisfactory response rate in clinical trials, hindered by immunosuppressive signals. To understand how microenvironmental characteristics alter immune homeostasis and limit immunotherapy efficacy in nasopharyngeal carcinoma, here we establish a multi-center single-cell cohort based on public data, containing 357,206 cells from 50 patient samples. We reveal that nasopharyngeal carcinoma cells enhance development and suppressive activity of regulatory T cells via CD70-CD27 interaction. CD70 blocking reverts Treg-mediated suppression and thus reinvigorate CD8+ T-cell immunity. Anti-CD70+ anti-PD-1 therapy is evaluated in xenograft-derived organoids and humanized mice, exhibiting an improved tumor-killing efficacy. Mechanistically, CD70 knockout inhibits a collective lipid signaling network in CD4+ naive and regulatory T cells involving mitochondrial integrity, cholesterol homeostasis, and fatty acid metabolism. Furthermore, ATAC-Seq delineates that CD70 is transcriptionally upregulated by NFKB2 via an Epstein-Barr virus-dependent epigenetic modification. Our findings identify CD70+ nasopharyngeal carcinoma cells as a metabolic switch that enforces the lipid-driven development, functional specialization and homeostasis of Tregs, leading to immune evasion. This study also demonstrates that CD70 blockade can act synergistically with anti-PD-1 treatment to reinvigorate T-cell immunity against nasopharyngeal carcinoma <span class="citation">(<a href="#ref-gong2023nasopharyngeal" role="doc-biblioref">Gong et al. 2023</a>)</span>.</p>
<div class="figure">
<img src="figs/spatialDatasets/NPC_Fig2_HTML.webp" alt="" />
<p class="caption">npc cancer fig2c</p>
</div>
<ol start="6" style="list-style-type: decimal">
<li>Prostate</li>
</ol>
<p><a href="https://www.nature.com/articles/s41586-022-05023-2">Spatially resolved clonal copy number alterations in benign and malignant tissue</a></p>
<p>Defining the transition from benign to malignant tissue is fundamental to improving early diagnosis of cancer1. Here we use a systematic approach to study spatial genome integrity in situ and describe previously unidentified clonal relationships. We used spatially resolved transcriptomics2 to infer spatial copy number variations in &gt;120,000 regions across multiple organs, in benign and malignant tissues. We demonstrate that genome-wide copy number variation reveals distinct clonal patterns within tumours and in nearby benign tissue using an organ-wide approach focused on the prostate. Our results suggest a model for how genomic instability arises in histologically benign tissue that may represent early events in cancer evolution. We highlight the power of capturing the molecular and spatial continuums in a tissue context and challenge the rationale for treatment paradigms, including focal therapy.<span class="citation">(<a href="#ref-erickson2022spatially" role="doc-biblioref">Erickson et al. 2022</a>)</span></p>
<div class="figure">
<img src="figs/spatialDatasets/prostate_Fig1_HTML.webp" alt="" />
<p class="caption">prostate cancer fig1</p>
</div>
<ol start="7" style="list-style-type: decimal">
<li>Skin</li>
</ol>
<p><a href="https://www.cell.com/cell/fulltext/S0092-8674(20)30672-3?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867420306723%3Fshowall%3Dtrue">Multimodal Analysis of Composition and Spatial Architecture in Human Squamous Cell Carcinoma</a></p>
<div class="figure">
<img src="figs/spatialDatasets/Skin%20Graphical%20Abstract.jpg" alt="" />
<p class="caption">prostate cancer fig1</p>
</div>
<p><strong>Highlights</strong></p>
<ul>
<li><p>Profiling of 10 human skin SCCs and matched normals via scRNA-seq, ST, and MIBI</p></li>
<li><p>Tumor-specific keratinocytes (TSKs) reside within a fibrovascular niche at leading edges</p></li>
<li><p>Distinct ligand-receptor and spatial niche associations for tumor and stromal cells.</p></li>
<li><p>Subpopulation essential tumorigenic gene networks defined by in vivo CRISPR screening</p></li>
</ul>
<p><strong>Summary</strong>
To define the cellular composition and architecture of cutaneous squamous cell carcinoma (cSCC), we combined single-cell RNA sequencing with spatial transcriptomics and multiplexed ion beam imaging from a series of human cSCCs and matched normal skin. cSCC exhibited four tumor subpopulations, three recapitulating normal epidermal states, and a tumor-specific keratinocyte (TSK) population unique to cancer, which localized to a fibrovascular niche. Integration of single-cell and spatial data mapped ligand-receptor networks to specific cell types, revealing TSK cells as a hub for intercellular communication. Multiple features of potential immunosuppression were observed, including T regulatory cell (Treg) co-localization with CD8 T cells in compartmentalized tumor stroma. Finally, single-cell characterization of human tumor xenografts and in vivo CRISPR screens identified essential roles for specific tumor subpopulation-enriched gene networks in tumorigenesis. These data define cSCC tumor and stromal cell subpopulations, the spatial niches where they interact, and the communicating gene networks that they engage in cancer<span class="citation">(<a href="#ref-ji2020multimodal" role="doc-biblioref">Ji et al. 2020</a>)</span>.</p>

</div>
<div id="d-biology" class="section level2 hasAnchor" number="6.8">
<h2><span class="header-section-number">6.8</span> 3D biology<a href="phd-research-topic-based.html#d-biology" class="anchor-section" aria-label="Anchor link to header"></a></h2>

</div>
<div id="sequencing-data-relative-knowledge" class="section level2 hasAnchor" number="6.9">
<h2><span class="header-section-number">6.9</span> Sequencing data relative Knowledge<a href="phd-research-topic-based.html#sequencing-data-relative-knowledge" class="anchor-section" aria-label="Anchor link to header"></a></h2>

</div>
<div id="image-data-relative-knowledge" class="section level2 hasAnchor" number="6.10">
<h2><span class="header-section-number">6.10</span> Image data relative Knowledge<a href="phd-research-topic-based.html#image-data-relative-knowledge" class="anchor-section" aria-label="Anchor link to header"></a></h2>

</div>
</div>
<h3>References<a href="references.html#references" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-elyanow2021starch" class="csl-entry">
Elyanow, Rebecca, Ron Zeira, Max Land, and Benjamin J Raphael. 2021. <span>“STARCH: Copy Number and Clone Inference from Spatial Transcriptomics Data.”</span> <em>Physical Biology</em> 18 (3): 035001.
</div>
<div id="ref-erickson2021spatial" class="csl-entry">
Erickson, Andrew, Emelie Berglund, Mengxiao He, Maja Marklund, Reza Mirzazadeh, Niklas Schultz, Ludvig Bergenstråhle, et al. 2021. <span>“The Spatial Landscape of Clonal Somatic Mutations in Benign and Malignant Tissue.”</span> <em>bioRxiv</em>.
</div>
<div id="ref-erickson2022spatially" class="csl-entry">
Erickson, Andrew, Mengxiao He, Emelie Berglund, Maja Marklund, Reza Mirzazadeh, Niklas Schultz, Linda Kvastad, et al. 2022. <span>“Spatially Resolved Clonal Copy Number Alterations in Benign and Malignant Tissue.”</span> <em>Nature</em> 608 (7922): 360–67.
</div>
<div id="ref-gong2023nasopharyngeal" class="csl-entry">
Gong, Lanqi, Jie Luo, Yu Zhang, Yuma Yang, Shanshan Li, Xiaona Fang, Baifeng Zhang, et al. 2023. <span>“Nasopharyngeal Carcinoma Cells Promote Regulatory t Cell Development and Suppressive Activity via Cd70-Cd27 Interaction.”</span> <em>Nature Communications</em> 14 (1): 1912.
</div>
<div id="ref-greenwald2024integrative" class="csl-entry">
Greenwald, Alissa C, Noam Galili Darnell, Rouven Hoefflin, Dor Simkin, Christopher W Mount, L Nicolas Gonzalez Castro, Yotam Harnik, et al. 2024. <span>“Integrative Spatial Analysis Reveals a Multi-Layered Organization of Glioblastoma.”</span> <em>Cell</em> 187 (10): 2485–2501.
</div>
<div id="ref-ji2020multimodal" class="csl-entry">
Ji, Andrew L, Adam J Rubin, Kim Thrane, Sizun Jiang, David L Reynolds, Robin M Meyers, Margaret G Guo, et al. 2020. <span>“Multimodal Analysis of Composition and Spatial Architecture in Human Squamous Cell Carcinoma.”</span> <em>Cell</em> 182 (2): 497–514.
</div>
<div id="ref-ravi2022spatially" class="csl-entry">
Ravi, Vidhya M, Paulina Will, Jan Kueckelhaus, Na Sun, Kevin Joseph, Henrike Salié, Lea Vollmer, et al. 2022. <span>“Spatially Resolved Multi-Omics Deciphers Bidirectional Tumor-Host Interdependence in Glioblastoma.”</span> <em>Cancer Cell</em> 40 (6): 639–55.
</div>
<div id="ref-ren2023spatial" class="csl-entry">
Ren, Yanming, Zongyao Huang, Lingling Zhou, Peng Xiao, Junwei Song, Ping He, Chuanxing Xie, et al. 2023. <span>“Spatial Transcriptomics Reveals Niche-Specific Enrichment and Vulnerabilities of Radial Glial Stem-Like Cells in Malignant Gliomas.”</span> <em>Nature Communications</em> 14 (1): 1028.
</div>
<div id="ref-wu2021comprehensive" class="csl-entry">
Wu, Rui, Wenbo Guo, Xinyao Qiu, Shicheng Wang, Chengjun Sui, Qiuyu Lian, Jianmin Wu, et al. 2021. <span>“Comprehensive Analysis of Spatial Architecture in Primary Liver Cancer.”</span> <em>Science Advances</em> 7 (51): eabg3750.
</div>
<div id="ref-zeng2022statistical" class="csl-entry">
Zeng, Zexian, Yawei Li, Yiming Li, and Yuan Luo. 2022. <span>“Statistical and Machine Learning Methods for Spatially Resolved Transcriptomics Data Analysis.”</span> <em>Genome Biology</em> 23 (1): 1–23.
</div>
<div id="ref-zhang2021supergnova" class="csl-entry">
Zhang, Yiliang, Qiongshi Lu, Yixuan Ye, Kunling Huang, Wei Liu, Yuchang Wu, Xiaoyuan Zhong, et al. 2021. <span>“SUPERGNOVA: Local Genetic Correlation Analysis Reveals Heterogeneous Etiologic Sharing of Complex Traits.”</span> <em>Genome Biology</em> 22 (1): 1–30. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02478-w">https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02478-w</a>.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="statmethods.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="proteomics.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["ReadingNotes.pdf", "ReadingNotes.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
