<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>References | ReadingNotes: Have you read today’s papers?</title>
  <meta name="description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  <meta name="generator" content="bookdown 0.43 and GitBook 2.6.7" />

  <meta property="og:title" content="References | ReadingNotes: Have you read today’s papers?" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | ReadingNotes: Have you read today’s papers?" />
  
  <meta name="twitter:description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  

<meta name="author" content="Rongting Huang rthuang@connect.hku.hk" />


<meta name="date" content="2025-07-04" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="method-of-the-year.html"/>

<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Reading Notes-Rongting</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="" data-path="about-the-author.html"><a href="about-the-author.html"><i class="fa fa-check"></i>About the author</a></li>
<li class="chapter" data-level="" data-path="research-topics.html"><a href="research-topics.html"><i class="fa fa-check"></i>Research topics</a></li>
<li class="chapter" data-level="" data-path="about-the-book.html"><a href="about-the-book.html"><i class="fa fa-check"></i>About the book</a></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#rongting-reading-notes"><i class="fa fa-check"></i><b>1.1</b> Rongting Reading Notes</a></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#about-this-notebook"><i class="fa fa-check"></i><b>1.2</b> <strong>About This Notebook</strong></a></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#topics-covered"><i class="fa fa-check"></i><b>1.3</b> <strong>Topics Covered</strong></a></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#how-to-use-this-notebook"><i class="fa fa-check"></i><b>1.4</b> <strong>How to Use This Notebook</strong></a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="sgcell.html"><a href="sgcell.html"><i class="fa fa-check"></i><b>2</b> SingleCell</a>
<ul>
<li class="chapter" data-level="2.1" data-path="sgcell.html"><a href="sgcell.html#technology"><i class="fa fa-check"></i><b>2.1</b> Technology</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="sgcell.html"><a href="sgcell.html#single-cell"><i class="fa fa-check"></i><b>2.1.1</b> single cell</a></li>
<li class="chapter" data-level="2.1.2" data-path="sgcell.html"><a href="sgcell.html#spatial-single-cell"><i class="fa fa-check"></i><b>2.1.2</b> spatial single cell</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="sgcell.html"><a href="sgcell.html#review"><i class="fa fa-check"></i><b>2.2</b> Review</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="sgcell.html"><a href="sgcell.html#review-2021-06"><i class="fa fa-check"></i><b>2.2.1</b> Review-2021-06</a></li>
<li class="chapter" data-level="2.2.2" data-path="sgcell.html"><a href="sgcell.html#review-2021-07"><i class="fa fa-check"></i><b>2.2.2</b> Review-2021-07</a></li>
<li class="chapter" data-level="2.2.3" data-path="sgcell.html"><a href="sgcell.html#review-2021-10"><i class="fa fa-check"></i><b>2.2.3</b> Review-2021-10</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="sgcell.html"><a href="sgcell.html#data-processing"><i class="fa fa-check"></i><b>2.3</b> Data processing</a></li>
<li class="chapter" data-level="2.4" data-path="sgcell.html"><a href="sgcell.html#clustering-methods"><i class="fa fa-check"></i><b>2.4</b> Clustering methods</a></li>
<li class="chapter" data-level="2.5" data-path="sgcell.html"><a href="sgcell.html#benchmarking"><i class="fa fa-check"></i><b>2.5</b> Benchmarking</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="technology-related.html"><a href="technology-related.html"><i class="fa fa-check"></i><b>3</b> Technology Related</a>
<ul>
<li class="chapter" data-level="3.1" data-path="technology-related.html"><a href="technology-related.html#scrna-seq-modelling"><i class="fa fa-check"></i><b>3.1</b> scRNA-seq modelling</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="technology-related.html"><a href="technology-related.html#nb-models-for-scrna-seq"><i class="fa fa-check"></i><b>3.1.1</b> NB models for scRNA-seq</a></li>
<li class="chapter" data-level="3.1.2" data-path="technology-related.html"><a href="technology-related.html#batch-effect-correction-methods-for-scrna-seq"><i class="fa fa-check"></i><b>3.1.2</b> Batch-effect correction methods for scRNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="technology-related.html"><a href="technology-related.html#hi-seq"><i class="fa fa-check"></i><b>3.2</b> Hi-Seq</a></li>
<li class="chapter" data-level="3.3" data-path="technology-related.html"><a href="technology-related.html#languages-and-compilers"><i class="fa fa-check"></i><b>3.3</b> languages and compilers</a></li>
<li class="chapter" data-level="3.4" data-path="technology-related.html"><a href="technology-related.html#singlecell-analysis-tools"><i class="fa fa-check"></i><b>3.4</b> singlecell analysis tools</a></li>
<li class="chapter" data-level="3.5" data-path="technology-related.html"><a href="technology-related.html#singlecell-analysis-sources"><i class="fa fa-check"></i><b>3.5</b> singlecell analysis sources</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="spatial-technology.html"><a href="spatial-technology.html"><i class="fa fa-check"></i><b>4</b> Spatial Technology</a>
<ul>
<li class="chapter" data-level="4.1" data-path="spatial-technology.html"><a href="spatial-technology.html#sequencing-based-st"><i class="fa fa-check"></i><b>4.1</b> Sequencing based ST</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="statmethods.html"><a href="statmethods.html"><i class="fa fa-check"></i><b>5</b> StatMethods</a>
<ul>
<li class="chapter" data-level="5.1" data-path="statmethods.html"><a href="statmethods.html#hmm-based-methods"><i class="fa fa-check"></i><b>5.1</b> HMM based methods</a></li>
<li class="chapter" data-level="5.2" data-path="statmethods.html"><a href="statmethods.html#em-based-methods"><i class="fa fa-check"></i><b>5.2</b> EM based methods</a></li>
<li class="chapter" data-level="5.3" data-path="statmethods.html"><a href="statmethods.html#vb-based-methods"><i class="fa fa-check"></i><b>5.3</b> VB based methods</a></li>
<li class="chapter" data-level="5.4" data-path="statmethods.html"><a href="statmethods.html#glm"><i class="fa fa-check"></i><b>5.4</b> GLM</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html"><i class="fa fa-check"></i><b>6</b> PhD Research topic based</a>
<ul>
<li class="chapter" data-level="6.1" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#cnv-calling"><i class="fa fa-check"></i><b>6.1</b> CNV calling</a>
<ul>
<li class="chapter" data-level="6.1.1" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#breaking-point-detection"><i class="fa fa-check"></i><b>6.1.1</b> breaking point detection</a></li>
<li class="chapter" data-level="6.1.2" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#related-research"><i class="fa fa-check"></i><b>6.1.2</b> Related research</a></li>
<li class="chapter" data-level="6.1.3" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#smoothing-strategies-in-rdr"><i class="fa fa-check"></i><b>6.1.3</b> Smoothing strategies in RDR</a></li>
<li class="chapter" data-level="6.1.4" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#pubmon"><i class="fa fa-check"></i><b>6.1.4</b> PUBMON</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#clonal-tree"><i class="fa fa-check"></i><b>6.2</b> Clonal Tree</a></li>
<li class="chapter" data-level="6.3" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#deconvolution"><i class="fa fa-check"></i><b>6.3</b> Deconvolution</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#deconvolution-of-bulk-tissue-and-spatial-transcriptomic-data"><i class="fa fa-check"></i><b>6.3.1</b> Deconvolution of bulk tissue and spatial transcriptomic data</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#omics-integration"><i class="fa fa-check"></i><b>6.4</b> Omics integration</a></li>
<li class="chapter" data-level="6.5" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#spatial-transcriptomics"><i class="fa fa-check"></i><b>6.5</b> Spatial transcriptomics</a>
<ul>
<li class="chapter" data-level="6.5.1" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#an-overview-of-spatial-omics-methods"><i class="fa fa-check"></i><b>6.5.1</b> An overview of spatial omics methods</a></li>
</ul></li>
<li class="chapter" data-level="6.6" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#spatial-transcriptomics-methods"><i class="fa fa-check"></i><b>6.6</b> Spatial transcriptomics Methods</a></li>
<li class="chapter" data-level="6.7" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#spatial-datasets-collection"><i class="fa fa-check"></i><b>6.7</b> Spatial Datasets Collection</a></li>
<li class="chapter" data-level="6.8" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#d-biology"><i class="fa fa-check"></i><b>6.8</b> 3D biology</a></li>
<li class="chapter" data-level="6.9" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#sequencing-data-relative-knowledge"><i class="fa fa-check"></i><b>6.9</b> Sequencing data relative Knowledge</a></li>
<li class="chapter" data-level="6.10" data-path="phd-research-topic-based.html"><a href="phd-research-topic-based.html#image-data-relative-knowledge"><i class="fa fa-check"></i><b>6.10</b> Image data relative Knowledge</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="proteomics.html"><a href="proteomics.html"><i class="fa fa-check"></i><b>7</b> Proteomics</a>
<ul>
<li class="chapter" data-level="7.1" data-path="proteomics.html"><a href="proteomics.html#ptm-cross-talk"><i class="fa fa-check"></i><b>7.1</b> PTM cross-talk</a></li>
<li class="chapter" data-level="7.2" data-path="proteomics.html"><a href="proteomics.html#ptm-cluster"><i class="fa fa-check"></i><b>7.2</b> PTM Cluster</a></li>
<li class="chapter" data-level="7.3" data-path="proteomics.html"><a href="proteomics.html#ptm-and-cancer"><i class="fa fa-check"></i><b>7.3</b> PTM and cancer</a></li>
<li class="chapter" data-level="7.4" data-path="proteomics.html"><a href="proteomics.html#protein-structure"><i class="fa fa-check"></i><b>7.4</b> Protein structure</a>
<ul>
<li class="chapter" data-level="7.4.1" data-path="proteomics.html"><a href="proteomics.html#protein-folding-prediction-background"><i class="fa fa-check"></i><b>7.4.1</b> Protein Folding Prediction Background</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="epigenomics.html"><a href="epigenomics.html"><i class="fa fa-check"></i><b>8</b> Epigenomics</a></li>
<li class="chapter" data-level="9" data-path="pubmon-1.html"><a href="pubmon-1.html"><i class="fa fa-check"></i><b>9</b> Pubmon</a>
<ul>
<li class="chapter" data-level="9.1" data-path="pubmon-1.html"><a href="pubmon-1.html#phd-year-1"><i class="fa fa-check"></i><b>9.1</b> 2020-2021 PhD year 1</a></li>
<li class="chapter" data-level="9.2" data-path="pubmon-1.html"><a href="pubmon-1.html#phd-year-2"><i class="fa fa-check"></i><b>9.2</b> 2021-2022 PhD year 2</a>
<ul>
<li class="chapter" data-level="9.2.1" data-path="pubmon-1.html"><a href="pubmon-1.html#resources"><i class="fa fa-check"></i><b>9.2.1</b> Resources</a></li>
<li class="chapter" data-level="9.2.2" data-path="pubmon-1.html"><a href="pubmon-1.html#ppi-topic"><i class="fa fa-check"></i><b>9.2.2</b> PPI topic</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="pubmon-1.html"><a href="pubmon-1.html#phd-year-4"><i class="fa fa-check"></i><b>9.3</b> 2023-2024 PhD year 4</a></li>
<li class="chapter" data-level="9.4" data-path="pubmon-1.html"><a href="pubmon-1.html#pre-postdoc"><i class="fa fa-check"></i><b>9.4</b> 2024-2025 Pre Postdoc</a></li>
<li class="chapter" data-level="9.5" data-path="pubmon-1.html"><a href="pubmon-1.html#section-1"><i class="fa fa-check"></i><b>9.5</b> 2025</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="method-of-the-year.html"><a href="method-of-the-year.html"><i class="fa fa-check"></i><b>10</b> Method of the Year</a>
<ul>
<li class="chapter" data-level="10.1" data-path="method-of-the-year.html"><a href="method-of-the-year.html#method-of-the-year-2022"><i class="fa fa-check"></i><b>10.1</b> Method of the Year 2022:</a></li>
<li class="chapter" data-level="10.2" data-path="method-of-the-year.html"><a href="method-of-the-year.html#method-of-the-year-2021-protein-structure-prediction"><i class="fa fa-check"></i><b>10.2</b> Method of the Year 2021: Protein structure prediction</a></li>
<li class="chapter" data-level="10.3" data-path="method-of-the-year.html"><a href="method-of-the-year.html#method-of-the-year-2020-spatially-resolved-transcriptomics"><i class="fa fa-check"></i><b>10.3</b> Method of the Year 2020: spatially resolved transcriptomics</a></li>
<li class="chapter" data-level="10.4" data-path="method-of-the-year.html"><a href="method-of-the-year.html#method-of-the-year-2019-single-cell-multimodal-omics"><i class="fa fa-check"></i><b>10.4</b> Method of the Year 2019: Single-cell multimodal omics</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">ReadingNotes: Have you read today’s papers?</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered hasAnchor">
<h1>References<a href="references.html#references" class="anchor-section" aria-label="Anchor link to header"></a></h1>

<div id="refs" class="references csl-bib-body hanging-indent" entry-spacing="0">
<div class="csl-entry">
Ahsan, Mian Umair, Qian Liu, Li Fang, and Kai Wang. 2021. <span>“NanoCaller for Accurate Detection of SNPs and Indels in Difficult-to-Map Regions from Long-Read Sequencing by Haplotype-Aware Deep Neural Networks.”</span> <em>Genome Biology</em> 22 (1): 1–33. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02472-2">https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02472-2</a>.
</div>
<div class="csl-entry">
Argelaguet, Ricard, Anna SE Cuomo, Oliver Stegle, and John C Marioni. 2021. <span>“Computational Principles and Challenges in Single-Cell Data Integration.”</span> <em>Nature Biotechnology</em>, 1–14. <a href="https://www.nature.com/articles/s41587-021-00895-7">https://www.nature.com/articles/s41587-021-00895-7</a>.
</div>
<div class="csl-entry">
Breda, Jérémie, Mihaela Zavolan, and Erik van Nimwegen. 2021. <span>“Bayesian Inference of Gene Expression States from Single-Cell RNA-Seq Data.”</span> <em>Nature Biotechnology</em>, 1–9.
</div>
<div class="csl-entry">
Elyanow, Rebecca, Ron Zeira, Max Land, and Benjamin J Raphael. 2021. <span>“STARCH: Copy Number and Clone Inference from Spatial Transcriptomics Data.”</span> <em>Physical Biology</em> 18 (3): 035001.
</div>
<div class="csl-entry">
Erickson, Andrew, Emelie Berglund, Mengxiao He, Maja Marklund, Reza Mirzazadeh, Niklas Schultz, Ludvig Bergenstråhle, et al. 2021. <span>“The Spatial Landscape of Clonal Somatic Mutations in Benign and Malignant Tissue.”</span> <em>bioRxiv</em>.
</div>
<div class="csl-entry">
Erickson, Andrew, Mengxiao He, Emelie Berglund, Maja Marklund, Reza Mirzazadeh, Niklas Schultz, Linda Kvastad, et al. 2022. <span>“Spatially Resolved Clonal Copy Number Alterations in Benign and Malignant Tissue.”</span> <em>Nature</em> 608 (7922): 360–67.
</div>
<div class="csl-entry">
Gao, Teng, Ruslan Soldatov, Hirak Sarkar, Adam Kurkiewicz, Evan Biederstedt, Po-Ru Loh, and Peter Kharchenko. 2022. <span>“Haplotype-Enhanced Inference of Somatic Copy Number Profiles from Single-Cell Transcriptomes.”</span> <em>bioRxiv</em>.
</div>
<div class="csl-entry">
Gong, Lanqi, Jie Luo, Yu Zhang, Yuma Yang, Shanshan Li, Xiaona Fang, Baifeng Zhang, et al. 2023. <span>“Nasopharyngeal Carcinoma Cells Promote Regulatory t Cell Development and Suppressive Activity via CD70-CD27 Interaction.”</span> <em>Nature Communications</em> 14 (1): 1912.
</div>
<div class="csl-entry">
Greenwald, Alissa C, Noam Galili Darnell, Rouven Hoefflin, Dor Simkin, Christopher W Mount, L Nicolas Gonzalez Castro, Yotam Harnik, et al. 2024. <span>“Integrative Spatial Analysis Reveals a Multi-Layered Organization of Glioblastoma.”</span> <em>Cell</em> 187 (10): 2485–2501.
</div>
<div class="csl-entry">
Gürsoy, Gamze, Nancy Lu, Sarah Wagner, and Mark Gerstein. 2021. <span>“Recovering Genotypes and Phenotypes Using Allele-Specific Genes.”</span> <em>Genome Biology</em> 22 (1): 1–9. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02477-x">https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02477-x</a>.
</div>
<div class="csl-entry">
Hafemeister, Christoph, and Rahul Satija. 2019. <span>“Normalization and Variance Stabilization of Single-Cell RNA-Seq Data Using Regularized Negative Binomial Regression.”</span> <em>Genome Biology</em> 20 (1): 1–15.
</div>
<div class="csl-entry">
Ji, Andrew L, Adam J Rubin, Kim Thrane, Sizun Jiang, David L Reynolds, Robin M Meyers, Margaret G Guo, et al. 2020. <span>“Multimodal Analysis of Composition and Spatial Architecture in Human Squamous Cell Carcinoma.”</span> <em>Cell</em> 182 (2): 497–514.
</div>
<div class="csl-entry">
Karlsson, Kasper, Moritz Przybilla, Hang Xu, Eran Kotler, Kremena Karagyozova, Alexandra Sockell, Katherine Liu, et al. 2022. <span>“Experimental Evolution in TP53 Deficient Gastric Organoids Recapitulates Tumorigenesis.”</span> <em>bioRxiv</em>.
</div>
<div class="csl-entry">
Kharchenko, Peter V. 2021. <span>“The Triumphs and Limitations of Computational Methods for scRNA-Seq.”</span> <em>Nature Methods</em>, 1–10.
</div>
<div class="csl-entry">
Kurtz, David M., Joanne Soo, Lyron Co Ting Keh, Stefan Alig, Jacob J. Chabon, Brian J. Sworder, Andre Schultz, et al. 2021. <span>“Enhanced Detection of Minimal Residual Disease by Targeted Sequencing of Phased Variants in Circulating Tumor DNA.”</span> <em>Nature Biotechnology</em>, July. <a href="https://doi.org/10.1038/s41587-021-00981-w">https://doi.org/10.1038/s41587-021-00981-w</a>.
</div>
<div class="csl-entry">
Liu, Longqi, Ao Chen, Yuxiang Li, Jan Mulder, Holger Heyn, and Xun Xu. 2024. <span>“Spatiotemporal Omics for Biology and Medicine.”</span> <em>Cell</em> 187 (17): 4488–4519.
</div>
<div class="csl-entry">
Love, Michael I, Wolfgang Huber, and Simon Anders. 2014. <span>“Moderated Estimation of Fold Change and Dispersion for RNA-Seq Data with DESeq2.”</span> <em>Genome Biology</em> 15 (12): 1–21.
</div>
<div class="csl-entry">
Luecken, Malte D, Maren Büttner, Kridsadakorn Chaichoompu, Anna Danese, Marta Interlandi, Michaela Fee Mueller, Daniel C Strobl, et al. 2021. <span>“Benchmarking Atlas-Level Data Integration in Single-Cell Genomics.”</span> <em>Nature Methods</em>, 1–10.
</div>
<div class="csl-entry">
Ma, Wenjing, Kenong Su, and Hao Wu. 2021. <span>“Evaluation of Some Aspects in Supervised Cell Type Identification for Single-Cell RNA-Seq: Classifier, Feature Selection, and Reference Construction.”</span> <em>Genome Biology</em> 22 (1): 1–23. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02480-2">https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02480-2</a>.
</div>
<div class="csl-entry">
McCarthy, Davis J, Yunshun Chen, and Gordon K Smyth. 2012. <span>“Differential Expression Analysis of Multifactor RNA-Seq Experiments with Respect to Biological Variation.”</span> <em>Nucleic Acids Research</em> 40 (10): 4288–97.
</div>
<div class="csl-entry">
Ravi, Vidhya M, Paulina Will, Jan Kueckelhaus, Na Sun, Kevin Joseph, Henrike Salié, Lea Vollmer, et al. 2022. <span>“Spatially Resolved Multi-Omics Deciphers Bidirectional Tumor-Host Interdependence in Glioblastoma.”</span> <em>Cancer Cell</em> 40 (6): 639–55.
</div>
<div class="csl-entry">
Ren, Yanming, Zongyao Huang, Lingling Zhou, Peng Xiao, Junwei Song, Ping He, Chuanxing Xie, et al. 2023. <span>“Spatial Transcriptomics Reveals Niche-Specific Enrichment and Vulnerabilities of Radial Glial Stem-Like Cells in Malignant Gliomas.”</span> <em>Nature Communications</em> 14 (1): 1028.
</div>
<div class="csl-entry">
Robinson, Mark D, Davis J McCarthy, and Gordon K Smyth. 2010. <span>“edgeR: A Bioconductor Package for Differential Expression Analysis of Digital Gene Expression Data.”</span> <em>Bioinformatics</em> 26 (1): 139–40.
</div>
<div class="csl-entry">
Shajii, Ariya, Ibrahim Numanagić, Alexander T Leighton, Haley Greenyer, Saman Amarasinghe, and Bonnie Berger. 2021. <span>“A Python-Based Programming Language for High-Performance Computational Genomics.”</span> <em>Nature Biotechnology</em>, 1–2. <a href="https://www.nature.com/articles/s41587-021-00985-6">https://www.nature.com/articles/s41587-021-00985-6</a>.
</div>
<div class="csl-entry">
Tran, Hoa Thi Nhu, Kok Siong Ang, Marion Chevrier, Xiaomeng Zhang, Nicole Yee Shin Lee, Michelle Goh, and Jinmiao Chen. 2020. <span>“A Benchmark of Batch-Effect Correction Methods for Single-Cell RNA Sequencing Data.”</span> <em>Genome Biology</em> 21 (1): 1–32. <a href="https://link.springer.com/article/10.1186/s13059-019-1850-9">https://link.springer.com/article/10.1186/s13059-019-1850-9</a>.
</div>
<div class="csl-entry">
Vallejos, Catalina A, John C Marioni, and Sylvia Richardson. 2015. <span>“BASiCS: Bayesian Analysis of Single-Cell Sequencing Data.”</span> <em>PLoS Computational Biology</em> 11 (6): e1004333.
</div>
<div class="csl-entry">
Wei, Runmin, Siyuan He, Shanshan Bai, Emi Sei, Min Hu, Alastair Thompson, Ken Chen, Savitri Krishnamurthy, and Nicholas E Navin. 2022. <span>“Spatial Charting of Single-Cell Transcriptomes in Tissues.”</span> <em>Nature Biotechnology</em>, 1–10.
</div>
<div class="csl-entry">
Williams, Marc J, Tyler Funnell, Ciara H O’Flanagan, Andrew McPherson, Sohrab Salehi, Ignacio Vázquez-Garcı́a, Farhia Kabeer, et al. 2021. <span>“Evolutionary Tracking of Cancer Haplotypes at Single-Cell Resolution.”</span> <em>bioRxiv</em>.
</div>
<div class="csl-entry">
Wu, Rui, Wenbo Guo, Xinyao Qiu, Shicheng Wang, Chengjun Sui, Qiuyu Lian, Jianmin Wu, et al. 2021. <span>“Comprehensive Analysis of Spatial Architecture in Primary Liver Cancer.”</span> <em>Science Advances</em> 7 (51): eabg3750.
</div>
<div class="csl-entry">
Zappia, Luke, and Fabian J Theis. 2021. <span>“Over 1000 Tools Reveal Trends in the Single-Cell RNA-Seq Analysis Landscape.”</span> <em>bioRxiv</em>.
</div>
<div class="csl-entry">
Zeng, Zexian, Yawei Li, Yiming Li, and Yuan Luo. 2022. <span>“Statistical and Machine Learning Methods for Spatially Resolved Transcriptomics Data Analysis.”</span> <em>Genome Biology</em> 23 (1): 1–23.
</div>
<div class="csl-entry">
Zhang, Yiliang, Qiongshi Lu, Yixuan Ye, Kunling Huang, Wei Liu, Yuchang Wu, Xiaoyuan Zhong, et al. 2021. <span>“SUPERGNOVA: Local Genetic Correlation Analysis Reveals Heterogeneous Etiologic Sharing of Complex Traits.”</span> <em>Genome Biology</em> 22 (1): 1–30. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02478-w">https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02478-w</a>.
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="method-of-the-year.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": null,
    "text": null
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": ["ReadingNotes.pdf", "ReadingNotes.epub"],
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "subsection"
  }
});
});
</script>

</body>

</html>
