<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 SingleCell | ReadingNotes: Have you read today’s papers?</title>
  <meta name="description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 SingleCell | ReadingNotes: Have you read today’s papers?" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 SingleCell | ReadingNotes: Have you read today’s papers?" />
  
  <meta name="twitter:description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  

<meta name="author" content="Rongting Huang rthuang@connect.hku.hk" />


<meta name="date" content="2021-11-01" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="intro.html"/>
<link rel="next" href="technology-related.html"/>
<script src="libs/header-attrs-2.8/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Reading Notes-Rongting</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="" data-path="about-the-author.html"><a href="about-the-author.html"><i class="fa fa-check"></i>About the author</a></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="sgcell.html"><a href="sgcell.html"><i class="fa fa-check"></i><b>2</b> SingleCell</a>
<ul>
<li class="chapter" data-level="2.1" data-path="sgcell.html"><a href="sgcell.html#technology"><i class="fa fa-check"></i><b>2.1</b> Technology</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="sgcell.html"><a href="sgcell.html#single-cell"><i class="fa fa-check"></i><b>2.1.1</b> single cell</a></li>
<li class="chapter" data-level="2.1.2" data-path="sgcell.html"><a href="sgcell.html#spatial-single-cell"><i class="fa fa-check"></i><b>2.1.2</b> spatial single cell</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="sgcell.html"><a href="sgcell.html#review"><i class="fa fa-check"></i><b>2.2</b> Review</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="sgcell.html"><a href="sgcell.html#review-2021-06"><i class="fa fa-check"></i><b>2.2.1</b> Review-2021-06</a></li>
<li class="chapter" data-level="2.2.2" data-path="sgcell.html"><a href="sgcell.html#review-2021-07"><i class="fa fa-check"></i><b>2.2.2</b> Review-2021-07</a></li>
<li class="chapter" data-level="2.2.3" data-path="sgcell.html"><a href="sgcell.html#review-2021-10"><i class="fa fa-check"></i><b>2.2.3</b> Review-2021-10</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="sgcell.html"><a href="sgcell.html#data-processing"><i class="fa fa-check"></i><b>2.3</b> Data processing</a></li>
<li class="chapter" data-level="2.4" data-path="sgcell.html"><a href="sgcell.html#clustering-methods"><i class="fa fa-check"></i><b>2.4</b> Clustering methods</a></li>
<li class="chapter" data-level="2.5" data-path="sgcell.html"><a href="sgcell.html#languages-and-compilers"><i class="fa fa-check"></i><b>2.5</b> languages and compilers</a></li>
<li class="chapter" data-level="2.6" data-path="sgcell.html"><a href="sgcell.html#singlecell-analysis-tools"><i class="fa fa-check"></i><b>2.6</b> singlecell analysis tools</a></li>
<li class="chapter" data-level="2.7" data-path="sgcell.html"><a href="sgcell.html#singlecell-analysis-sources"><i class="fa fa-check"></i><b>2.7</b> singlecell analysis sources</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="technology-related.html"><a href="technology-related.html"><i class="fa fa-check"></i><b>3</b> Technology Related</a>
<ul>
<li class="chapter" data-level="3.1" data-path="technology-related.html"><a href="technology-related.html#scrna-seq-modelling"><i class="fa fa-check"></i><b>3.1</b> scRNA-seq modelling</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="technology-related.html"><a href="technology-related.html#nb-models-for-scrna-seq"><i class="fa fa-check"></i><b>3.1.1</b> NB models for scRNA-seq</a></li>
<li class="chapter" data-level="3.1.2" data-path="technology-related.html"><a href="technology-related.html#batch-effect-correction-methods-for-scrna-seq"><i class="fa fa-check"></i><b>3.1.2</b> Batch-effect correction methods for scRNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="technology-related.html"><a href="technology-related.html#hi-seq"><i class="fa fa-check"></i><b>3.2</b> Hi-Seq</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="statmethods.html"><a href="statmethods.html"><i class="fa fa-check"></i><b>4</b> StatMethods</a>
<ul>
<li class="chapter" data-level="4.1" data-path="statmethods.html"><a href="statmethods.html#hmm-based-methods"><i class="fa fa-check"></i><b>4.1</b> HMM based methods</a></li>
<li class="chapter" data-level="4.2" data-path="statmethods.html"><a href="statmethods.html#em-based-methods"><i class="fa fa-check"></i><b>4.2</b> EM based methods</a></li>
<li class="chapter" data-level="4.3" data-path="statmethods.html"><a href="statmethods.html#vb-based-methods"><i class="fa fa-check"></i><b>4.3</b> VB based methods</a></li>
<li class="chapter" data-level="4.4" data-path="statmethods.html"><a href="statmethods.html#glm"><i class="fa fa-check"></i><b>4.4</b> GLM</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="research-topic-based.html"><a href="research-topic-based.html"><i class="fa fa-check"></i><b>5</b> Research topic based</a>
<ul>
<li class="chapter" data-level="5.1" data-path="research-topic-based.html"><a href="research-topic-based.html#cnv-calling"><i class="fa fa-check"></i><b>5.1</b> CNV calling</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="research-topic-based.html"><a href="research-topic-based.html#breaking-point-detection"><i class="fa fa-check"></i><b>5.1.1</b> breaking point detection</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="research-topic-based.html"><a href="research-topic-based.html#clonal-tree"><i class="fa fa-check"></i><b>5.2</b> Clonal Tree</a></li>
<li class="chapter" data-level="5.3" data-path="research-topic-based.html"><a href="research-topic-based.html#deconvolution"><i class="fa fa-check"></i><b>5.3</b> Deconvolution</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="research-topic-based.html"><a href="research-topic-based.html#deconvolution-of-bulk-tissue-and-spatial-transcriptomic-data"><i class="fa fa-check"></i><b>5.3.1</b> Deconvolution of bulk tissue and spatial transcriptomic data</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="research-topic-based.html"><a href="research-topic-based.html#omics-integration"><i class="fa fa-check"></i><b>5.4</b> Omics integration</a></li>
<li class="chapter" data-level="5.5" data-path="research-topic-based.html"><a href="research-topic-based.html#spatial-transcriptomics"><i class="fa fa-check"></i><b>5.5</b> Spatial transcriptomics</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="proteomics.html"><a href="proteomics.html"><i class="fa fa-check"></i><b>6</b> Proteomics</a>
<ul>
<li class="chapter" data-level="6.1" data-path="proteomics.html"><a href="proteomics.html#ptm-cross-talk"><i class="fa fa-check"></i><b>6.1</b> PTM cross-talk</a></li>
<li class="chapter" data-level="6.2" data-path="proteomics.html"><a href="proteomics.html#ptm-cluster"><i class="fa fa-check"></i><b>6.2</b> PTM Cluster</a></li>
<li class="chapter" data-level="6.3" data-path="proteomics.html"><a href="proteomics.html#section-1"><i class="fa fa-check"></i><b>6.3</b> </a></li>
<li class="chapter" data-level="6.4" data-path="proteomics.html"><a href="proteomics.html#protein-structure"><i class="fa fa-check"></i><b>6.4</b> Protein structure</a>
<ul>
<li class="chapter" data-level="6.4.1" data-path="proteomics.html"><a href="proteomics.html#protein-folding-prediction-background"><i class="fa fa-check"></i><b>6.4.1</b> Protein Folding Prediction Background</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="pubmon.html"><a href="pubmon.html"><i class="fa fa-check"></i><b>7</b> Pubmon</a>
<ul>
<li class="chapter" data-level="7.1" data-path="pubmon.html"><a href="pubmon.html#phd-year-1"><i class="fa fa-check"></i><b>7.1</b> 2020-2021 PhD year 1</a></li>
<li class="chapter" data-level="7.2" data-path="pubmon.html"><a href="pubmon.html#phd-year-2"><i class="fa fa-check"></i><b>7.2</b> 2021-2022 PhD year 2</a>
<ul>
<li class="chapter" data-level="7.2.1" data-path="pubmon.html"><a href="pubmon.html#resources"><i class="fa fa-check"></i><b>7.2.1</b> Resources</a></li>
<li class="chapter" data-level="7.2.2" data-path="pubmon.html"><a href="pubmon.html#ppi-topic"><i class="fa fa-check"></i><b>7.2.2</b> PPI topic</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">ReadingNotes: Have you read today’s papers?</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="sgcell" class="section level1" number="2">
<h1><span class="header-section-number">Chapter 2</span> SingleCell</h1>
<div id="technology" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Technology</h2>
<div id="single-cell" class="section level3" number="2.1.1">
<h3><span class="header-section-number">2.1.1</span> single cell</h3>
<ul>
<li>scDNA</li>
<li>scRNA</li>
<li>scATAC</li>
</ul>
</div>
<div id="spatial-single-cell" class="section level3" number="2.1.2">
<h3><span class="header-section-number">2.1.2</span> spatial single cell</h3>
</div>
</div>
<div id="review" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Review</h2>
<div id="review-2021-06" class="section level3" number="2.2.1">
<h3><span class="header-section-number">2.2.1</span> Review-2021-06</h3>
<p><a href="https://www.nature.com/articles/s41592-021-01171-x">The triumphs and limitations of computational methods for scRNA-seq</a><span class="citation">(Kharchenko <a href="#ref-kharchenko2021triumphs" role="doc-biblioref">2021</a>)</span></p>
<p>Though computational approaches vary, most formulate (1) a statistical model of the measurement, (2) a representation of the data in reduced dimensions, and (3) an approximation of the expression manifold (Box 2), with a set of discrete transcriptional subpopulations being the simplest and the most common approximation. The problems motivating these steps, and the specific solutions and their assumptions, are the subject of this review.</p>
<p><img src="figs/singleCell/preprocessing_review1.jpg" alt="Key preprocessing steps in single-cell RNA-seq analysis" />
<a href="https://www.nature.com/articles/s41592-021-01171-x/figures/1">Key preprocessing steps in single-cell RNA-seq analysis</a></p>
<div class="figure">
<img src="figs/singleCell/key_analysis_review2.jpg" alt="" />
<p class="caption">Key analysis steps in single-cell RNA-seq analysis</p>
</div>
<p><a href="https://www.nature.com/articles/s41592-021-01171-x/figures/2">Key analysis steps in single-cell RNA-seq analysis</a></p>
<p><strong>Box 1 Select software tools</strong>
Tools for alignment, barcode correction, count matrix estimation, and quality control include:</p>
<ul>
<li><p>CellRanger (<a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/installation" class="uri">https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/installation</a>): supports 10x Chromium datasets (commercial product)</p></li>
<li><p>dropEst (<a href="https://github.com/hms-dbmi/dropEst" class="uri">https://github.com/hms-dbmi/dropEst</a>): supports multiple droplet-based protocols</p></li>
<li><p>STAR (<a href="https://github.com/alexdobin/STAR" class="uri">https://github.com/alexdobin/STAR</a>): aligner (used internally by CellRanger and dropEst), also has built-in options for count matrix estimation</p></li>
<li><p>Optimus (<a href="https://data.humancellatlas.org/pipelines/optimus-workflow" class="uri">https://data.humancellatlas.org/pipelines/optimus-workflow</a>): supports 10x Chromium v2 and v3 datasets, designed for Human Cell Atlas</p></li>
<li><p>Kallisto/bustools (<a href="https://www.kallistobus.tools" class="uri">https://www.kallistobus.tools</a>): fast processing using pseudoalignment</p></li>
</ul>
<p>Cell filter and doublet identification tools include:</p>
<ul>
<li><p>EmptyDrops (<a href="https://rdrr.io/github/MarioniLab/DropletUtils/man/emptyDrops.html" class="uri">https://rdrr.io/github/MarioniLab/DropletUtils/man/emptyDrops.html</a>): uses a classifier to distinguish ‘empty’ cells that look like the low-end tail of the cell size distribution</p></li>
<li><p>Scrublet (<a href="https://github.com/AllonKleinLab/scrublet" class="uri">https://github.com/AllonKleinLab/scrublet</a>): python-based, doublet simulation and doublet scoring</p></li>
<li><p>doubletFinder (<a href="https://github.com/chris-mcginnis-ucsf/DoubletFinder" class="uri">https://github.com/chris-mcginnis-ucsf/DoubletFinder</a>): R-based, doublet simulation and doublet scoring</p></li>
<li><p>scds (<a href="https://github.com/kostkalab/scds" class="uri">https://github.com/kostkalab/scds</a>): fast doublet scoring implementation</p></li>
</ul>
<p>Tools for normalization, dimensionality reduction, and clustering and differential expression include:</p>
<ul>
<li><p>Seurat (<a href="https://satijalab.org/seurat/" class="uri">https://satijalab.org/seurat/</a>): the most popular analysis toolkit, R-based</p></li>
<li><p>scanpy (<a href="https://github.com/theislab/scanpy" class="uri">https://github.com/theislab/scanpy</a>): the most popular python-based toolkit</p></li>
<li><p>scVI (<a href="https://github.com/YosefLab/scVI" class="uri">https://github.com/YosefLab/scVI</a>): latent space identification using variational neural net</p></li>
<li><p>pagoda2 (<a href="https://github.com/hms-dbmi/pagoda2" class="uri">https://github.com/hms-dbmi/pagoda2</a>): fast, R-based processing</p></li>
<li><p>SAUCIE (<a href="https://www.krishnaswamylab.org/projects/saucie" class="uri">https://www.krishnaswamylab.org/projects/saucie</a>): a neural-net-based dimensionality reduction, using maximal mean discrepancy penalty</p></li>
</ul>
<p>Tools for trajectory fitting include:</p>
<ul>
<li><p>Monocle3 (<a href="https://cole-trapnell-lab.github.io/monocle3/" class="uri">https://cole-trapnell-lab.github.io/monocle3/</a>): third iteration of the Monocle package, including updated tree utilities</p></li>
<li><p>Slingshot (<a href="https://github.com/kstreet13/slingshot" class="uri">https://github.com/kstreet13/slingshot</a>): tree fitting with improved pseudotime estimation</p></li>
<li><p>PAGA (<a href="https://github.com/theislab/paga" class="uri">https://github.com/theislab/paga</a>): tree/graph fitting approach combined with cell aggregation, also supports cluster-based velocity estimates</p></li>
<li><p>Wishbone (<a href="https://dpeerlab.github.io/dpeerlab-website/wishbone.html" class="uri">https://dpeerlab.github.io/dpeerlab-website/wishbone.html</a>): a bifurcation analysis method</p></li>
<li><p>Destiny, DPT (<a href="https://github.com/theislab/destiny/" class="uri">https://github.com/theislab/destiny/</a>): dimensionality reduction and trajectory fitting using diffusion maps82</p></li>
</ul>
<p>Tools for velocity estimation include:</p>
<ul>
<li><p>velocyto (<a href="http://velocyto.org/" class="uri">http://velocyto.org/</a>): reference python/R implementation</p></li>
<li><p>scVelo (<a href="https://scvelo.readthedocs.io/" class="uri">https://scvelo.readthedocs.io/</a>): new implementation using curve-based phase portrait fit</p></li>
</ul>
<div id="statistical-view-of-a-cell" class="section level4" number="2.2.1.1">
<h4><span class="header-section-number">2.2.1.1</span> Statistical view of a cell</h4>
</div>
<div id="comparing-transcriptional-states" class="section level4" number="2.2.1.2">
<h4><span class="header-section-number">2.2.1.2</span> Comparing transcriptional states</h4>
</div>
<div id="the-quest-for-reduced-dimensions" class="section level4" number="2.2.1.3">
<h4><span class="header-section-number">2.2.1.3</span> The quest for reduced dimensions</h4>
</div>
<div id="section" class="section level4" number="2.2.1.4">
<h4><span class="header-section-number">2.2.1.4</span> </h4>
<div class="figure">
<img src="figs/singleCell/scRNA-seq%20basics.jpg" alt="" />
<p class="caption">scRNA-seq basics</p>
</div>
<p><a href="https://www.nature.com/articles/s41592-021-01171-x/figures/3">scRNA-seq basics</a></p>
<p>a, Beating Moore’s law. The number of cells measured by landmark scRNA-seq datasets over years (red), compared with the increase in the CPU transistor counts (black). The set of all published scRNA-seq studies83 is shown with small red dots. The estimated number of cells in a human body is shown by a green dashed line.
b, Shallow coverage of each cell can be compensated for by measuring more cells. The ability to distinguish two cell populations, assessed by the area under the receiver operating characteristic curve (ROC AUC) measure, is shown as a function of the number of measured cells (x axis) and the mean cell depth (y axis). Examples of three different simulations (1-3) within different parts of this design parameter space are shown on PCA projections.
c, Probabilistic view of scRNA-seq estimates. Posterior probability of IL32 gene expression magnitude is shown for five cells from two different CD8+ T cell populations (red and blue, thin lines). Joint posteriors assessing the mean expression magnitude within each subpopulation are shown by thick dashed lines.
d, Comparing CD4+ T cells and CD14+ monocytes, the plot shows the number (y axis, left) and the fraction (y axis, right) of the genes passing a 1% statistical significance threshold for differential expression (DE) as a function of the number of cells compared from each population (x axis).
e, The scatter plot shows for each gene (dots) the mean (x axis) and variance (y axis) of the normalized UMI counts (CPM, counts per million) in CD4+ T cells. The Poisson expected value is shown in green, with a quadratic-based negative binomial fit shown in red. f-i, Variance normalization and most variable genes.
f, A t-SNE embedding of a primary peripheral blood mononuclear cell (PBMC) dataset with cell annotations. NK, natural killer, separated into CD56 bright and dim subsets. pDC, plasmacytoid dendritic cell.
g, Mean-variance relationship of different genes (dots) in the PBMC dataset is shown for log-transformed expression estimates. The genome-wide relationship, as captured by smoothed regression, is shown by the blue line. Genes whose variance is significantly higher than the genome-wide trend are shown as red dots.
h, Residual variance is shown for the top 5,000 overdispersed genes, ordered by the statistical significance (x axis).
i, Expression pattern of several example genes, with circles highlighting the subpopulations distinguished by the genes.
j, Distribution of normalized expression magnitudes (CPM) for the CTSH gene across all CD14+ monocytes is shown on the linear scale (top) and after log transformation (bottom) with a pseudocount.</p>
</div>
</div>
<div id="review-2021-07" class="section level3" number="2.2.2">
<h3><span class="header-section-number">2.2.2</span> Review-2021-07</h3>
<p><a href="https://www.nature.com/articles/s41587-021-00895-7">Computational principles and challenges in single-cell data integration</a><span class="citation">(Argelaguet et al. <a href="#ref-argelaguet2021computational" role="doc-biblioref">2021</a>)</span></p>
</div>
<div id="review-2021-10" class="section level3" number="2.2.3">
<h3><span class="header-section-number">2.2.3</span> Review-2021-10</h3>
<p><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02519-4">Over 1000 tools reveal trends in the single-cell RNA-seq analysis landscape</a><span class="citation">(Zappia and Theis <a href="#ref-zappia2021over" role="doc-biblioref">2021</a>)</span></p>
<p>Single-cell gene expression measurements are cell type-specific (unlike DNA), more easily interpretable (compared to epigenetic modalities), and scalable to thousands of features (unlike antibody-based protein measurements) and thousands of cells. These features mean that scRNA-seq can be used as an anchor, often measured in parallel and used to link other modalities.</p>
<p><a href="https://github.com/Rongtingting/awesome-single-cell">The Awesome Single Cell repository</a> is a community-curated list of software packages, resources, researchers, and publications for various single-cell technologies and <a href="https://docs.google.com/spreadsheets/d/1IPe2ozb1Mny8sLvJaSE57RJr3oruiBoSudAVhSH-O8M/edit?pli=1#gid=237186399">Albert Villela’s SingleCell Omics spreadsheet</a> tracks a range of information including technologies, companies, and software tools.</p>
<p>The scRNA-tools database focuses specifically on the cataloging and manual curation of software tools for analyzing scRNA-seq data [23]. When tools become available (usually through a bioRxiv preprint), we classify them according to the analysis tasks they can be used for and record information such as associated preprints and publications, software licenses, code location, software repositories, and a short description. Most tools are added to the database within 30 days of the first preprint or publication (Additional file 1: Figure S1). All the recorded information is publicly available in an interactive format at <a href="https://www.scrna-tools.org/" class="uri">https://www.scrna-tools.org/</a> [24]. As the number of tools in the database has moved past 1000, we have taken this opportunity to provide an update on the current state of the database and explore trends in scRNA-seq analysis across the past 5 years. We find that the focus of tool developers has moved on from continuous ordering of cells to methods for integrating samples and classifying cells. The database also shows us that more new tools are built using Python while the relative usage of R is declining. We also examine the role of open science in the development of the field and find that open source practices lead to increased citations. While the scRNA-tools database does not record every scRNA-seq analysis tool, the large proportion it does include over the history of what is still a young field make these analyses possible and a reasonable estimate of trends across all tools.</p>
</div>
</div>
<div id="data-processing" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Data processing</h2>
<p><a href="https://www.nature.com/articles/s41587-021-00875-x">Bayesian inference of gene expression states from single-cell RNA-seq data</a><span class="citation">(Breda, Zavolan, and Nimwegen <a href="#ref-breda2021bayesian" role="doc-biblioref">2021</a>)</span></p>
<p><a href="https://www.nature.com/articles/s41587-021-00875-x/figures/1">summary of sanity approach</a></p>
<div class="figure">
<img src="figs/singleCell/summary_Sanity.jpg" alt="" />
<p class="caption">summary of sanity approach</p>
</div>
<div class="figure">
<img src="figs/singleCell/sanity_notes.png" alt="" />
<p class="caption">sanity notes</p>
</div>
</div>
<div id="clustering-methods" class="section level2" number="2.4">
<h2><span class="header-section-number">2.4</span> Clustering methods</h2>

</div>
<div id="languages-and-compilers" class="section level2" number="2.5">
<h2><span class="header-section-number">2.5</span> languages and compilers</h2>
<ul>
<li>Seq</li>
</ul>
<p><a href="https://www.nature.com/articles/s41587-021-00985-6">A Python-based programming language for high-performance computational genomics</a><span class="citation">(Shajii et al. <a href="#ref-shajii2021python" role="doc-biblioref">2021</a>)</span></p>
<div class="figure">
<img src="figs/computationalBio/The%20Seq%20programming%20language.jpg" alt="" />
<p class="caption">The Seq programming language.</p>
</div>
<p>a, Conceptual comparison of Seq, Python and C++. Seq combines the high performance of C++ with the programming ease and clarity of Python, by virtue of domain-specific compiler optimizations that are hidden from the user. b, Example Seq code for a simple k-mer-based read mapper. c, Schematic of standard genomics pipeline and those state-of-the-art tools compared to Seq.</p>
<p>To demonstrate Seq’s versatility, we reimplemented eight popular genomics tools in Seq, spanning key tasks in the genomics analysis pipeline (Fig. 1c and Supplementary Note 2), such as the finding of super-maximal exact matches, or SMEMs (BWA-MEM13), genome homology table construction (CORA14), Hamming distance-based all-mapping (mrsFAST15), long-read alignment (minimap216), <strong>single-cell data preprocessing (UMI-tools17)</strong>, SAM/BAM post-processing (GATK18), global sequence alignment (AVID19) and <strong>haplotype phasing (Haptree-X20,21)</strong>.</p>
<p><a href="https://github.com/seq-lang/seq-benchmarks/tree/master/seq-nbt#haptree-x-haplotype-phasing">Hap Tree-X</a></p>
</div>
<div id="singlecell-analysis-tools" class="section level2" number="2.6">
<h2><span class="header-section-number">2.6</span> singlecell analysis tools</h2>
<ul>
<li><p>scanpy
<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1382-0">SCANPY: large-scale single-cell gene expression data analysis</a></p></li>
<li><p>seurat</p></li>
</ul>
</div>
<div id="singlecell-analysis-sources" class="section level2" number="2.7">
<h2><span class="header-section-number">2.7</span> singlecell analysis sources</h2>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references hanging-indent">
<div id="ref-argelaguet2021computational">
<p>Argelaguet, Ricard, Anna SE Cuomo, Oliver Stegle, and John C Marioni. 2021. “Computational Principles and Challenges in Single-Cell Data Integration.” <em>Nature Biotechnology</em>, 1–14. <a href="https://www.nature.com/articles/s41587-021-00895-7">https://www.nature.com/articles/s41587-021-00895-7</a>.</p>
</div>
<div id="ref-breda2021bayesian">
<p>Breda, Jérémie, Mihaela Zavolan, and Erik van Nimwegen. 2021. “Bayesian Inference of Gene Expression States from Single-Cell Rna-Seq Data.” <em>Nature Biotechnology</em>, 1–9. <a href="https://www.nature.com/articles/s41587-021-00875-x">https://www.nature.com/articles/s41587-021-00875-x</a>.</p>
</div>
<div id="ref-kharchenko2021triumphs">
<p>Kharchenko, Peter V. 2021. “The Triumphs and Limitations of Computational Methods for scRNA-Seq.” <em>Nature Methods</em>, 1–10.</p>
</div>
<div id="ref-shajii2021python">
<p>Shajii, Ariya, Ibrahim Numanagić, Alexander T Leighton, Haley Greenyer, Saman Amarasinghe, and Bonnie Berger. 2021. “A Python-Based Programming Language for High-Performance Computational Genomics.” <em>Nature Biotechnology</em>, 1–2. <a href="https://www.nature.com/articles/s41587-021-00985-6">https://www.nature.com/articles/s41587-021-00985-6</a>.</p>
</div>
<div id="ref-zappia2021over">
<p>Zappia, Luke, and Fabian J Theis. 2021. “Over 1000 Tools Reveal Trends in the Single-Cell Rna-Seq Analysis Landscape.” <em>bioRxiv</em>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="intro.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="technology-related.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["ReadingNotes.pdf", "ReadingNotes.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
