<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Technology Related | ReadingNotes: Have you read today’s papers?</title>
  <meta name="description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Technology Related | ReadingNotes: Have you read today’s papers?" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Technology Related | ReadingNotes: Have you read today’s papers?" />
  
  <meta name="twitter:description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  

<meta name="author" content="Rongting Huang rthuang@connect.hku.hk" />


<meta name="date" content="2021-10-14" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="sgcell.html"/>
<link rel="next" href="statmethods.html"/>
<script src="libs/header-attrs-2.8/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Reading Notes-Rongting</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="" data-path="about-the-author.html"><a href="about-the-author.html"><i class="fa fa-check"></i>About the author</a></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="sgcell.html"><a href="sgcell.html"><i class="fa fa-check"></i><b>2</b> SingleCell</a>
<ul>
<li class="chapter" data-level="2.1" data-path="sgcell.html"><a href="sgcell.html#technology"><i class="fa fa-check"></i><b>2.1</b> Technology</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="sgcell.html"><a href="sgcell.html#single-cell"><i class="fa fa-check"></i><b>2.1.1</b> single cell</a></li>
<li class="chapter" data-level="2.1.2" data-path="sgcell.html"><a href="sgcell.html#spatial-single-cell"><i class="fa fa-check"></i><b>2.1.2</b> spatial single cell</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="sgcell.html"><a href="sgcell.html#review"><i class="fa fa-check"></i><b>2.2</b> Review</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="sgcell.html"><a href="sgcell.html#review-2021-06"><i class="fa fa-check"></i><b>2.2.1</b> Review-2021-06</a></li>
<li class="chapter" data-level="2.2.2" data-path="sgcell.html"><a href="sgcell.html#review-2021-07"><i class="fa fa-check"></i><b>2.2.2</b> Review-2021-07</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="sgcell.html"><a href="sgcell.html#data-processing"><i class="fa fa-check"></i><b>2.3</b> Data processing</a></li>
<li class="chapter" data-level="2.4" data-path="sgcell.html"><a href="sgcell.html#clustering-methods"><i class="fa fa-check"></i><b>2.4</b> Clustering methods</a></li>
<li class="chapter" data-level="2.5" data-path="sgcell.html"><a href="sgcell.html#languages-and-compilers"><i class="fa fa-check"></i><b>2.5</b> languages and compilers</a></li>
<li class="chapter" data-level="2.6" data-path="sgcell.html"><a href="sgcell.html#singlecell-analysis-tools"><i class="fa fa-check"></i><b>2.6</b> singlecell analysis tools</a></li>
<li class="chapter" data-level="2.7" data-path="sgcell.html"><a href="sgcell.html#singlecell-analysis-sources"><i class="fa fa-check"></i><b>2.7</b> singlecell analysis sources</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="technology-related.html"><a href="technology-related.html"><i class="fa fa-check"></i><b>3</b> Technology Related</a>
<ul>
<li class="chapter" data-level="3.1" data-path="technology-related.html"><a href="technology-related.html#scrna-seq-modelling"><i class="fa fa-check"></i><b>3.1</b> scRNA-seq modelling</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="technology-related.html"><a href="technology-related.html#nb-models-for-scrna-seq"><i class="fa fa-check"></i><b>3.1.1</b> NB models for scRNA-seq</a></li>
<li class="chapter" data-level="3.1.2" data-path="technology-related.html"><a href="technology-related.html#batch-effect-correction-methods-for-scrna-seq"><i class="fa fa-check"></i><b>3.1.2</b> Batch-effect correction methods for scRNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="technology-related.html"><a href="technology-related.html#hi-seq"><i class="fa fa-check"></i><b>3.2</b> Hi-Seq</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="statmethods.html"><a href="statmethods.html"><i class="fa fa-check"></i><b>4</b> StatMethods</a>
<ul>
<li class="chapter" data-level="4.1" data-path="statmethods.html"><a href="statmethods.html#hmm-based-methods"><i class="fa fa-check"></i><b>4.1</b> HMM based methods</a></li>
<li class="chapter" data-level="4.2" data-path="statmethods.html"><a href="statmethods.html#em-based-methods"><i class="fa fa-check"></i><b>4.2</b> EM based methods</a></li>
<li class="chapter" data-level="4.3" data-path="statmethods.html"><a href="statmethods.html#vb-based-methods"><i class="fa fa-check"></i><b>4.3</b> VB based methods</a></li>
<li class="chapter" data-level="4.4" data-path="statmethods.html"><a href="statmethods.html#glm"><i class="fa fa-check"></i><b>4.4</b> GLM</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="research-topic-based.html"><a href="research-topic-based.html"><i class="fa fa-check"></i><b>5</b> Research topic based</a>
<ul>
<li class="chapter" data-level="5.1" data-path="research-topic-based.html"><a href="research-topic-based.html#cnv-calling"><i class="fa fa-check"></i><b>5.1</b> CNV calling</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="research-topic-based.html"><a href="research-topic-based.html#breaking-point-detection"><i class="fa fa-check"></i><b>5.1.1</b> breaking point detection</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="research-topic-based.html"><a href="research-topic-based.html#clonal-tree"><i class="fa fa-check"></i><b>5.2</b> Clonal Tree</a></li>
<li class="chapter" data-level="5.3" data-path="research-topic-based.html"><a href="research-topic-based.html#deconvolution"><i class="fa fa-check"></i><b>5.3</b> Deconvolution</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="research-topic-based.html"><a href="research-topic-based.html#deconvolution-of-bulk-tissue-and-spatial-transcriptomic-data"><i class="fa fa-check"></i><b>5.3.1</b> Deconvolution of bulk tissue and spatial transcriptomic data</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="research-topic-based.html"><a href="research-topic-based.html#omics-integration"><i class="fa fa-check"></i><b>5.4</b> Omics integration</a></li>
<li class="chapter" data-level="5.5" data-path="research-topic-based.html"><a href="research-topic-based.html#spatial-transcriptomics"><i class="fa fa-check"></i><b>5.5</b> Spatial transcriptomics</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="proteomics.html"><a href="proteomics.html"><i class="fa fa-check"></i><b>6</b> Proteomics</a>
<ul>
<li class="chapter" data-level="6.1" data-path="proteomics.html"><a href="proteomics.html#ptm-cross-talk"><i class="fa fa-check"></i><b>6.1</b> PTM cross-talk</a></li>
<li class="chapter" data-level="6.2" data-path="proteomics.html"><a href="proteomics.html#ptm-cluster"><i class="fa fa-check"></i><b>6.2</b> PTM Cluster</a></li>
<li class="chapter" data-level="6.3" data-path="proteomics.html"><a href="proteomics.html#section-1"><i class="fa fa-check"></i><b>6.3</b> </a></li>
<li class="chapter" data-level="6.4" data-path="proteomics.html"><a href="proteomics.html#protein-structure"><i class="fa fa-check"></i><b>6.4</b> Protein structure</a>
<ul>
<li class="chapter" data-level="6.4.1" data-path="proteomics.html"><a href="proteomics.html#protein-folding-prediction-background"><i class="fa fa-check"></i><b>6.4.1</b> Protein Folding Prediction Background</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="pubmon.html"><a href="pubmon.html"><i class="fa fa-check"></i><b>7</b> Pubmon</a>
<ul>
<li class="chapter" data-level="7.1" data-path="pubmon.html"><a href="pubmon.html#phd-year-1"><i class="fa fa-check"></i><b>7.1</b> 2020-2021 PhD year 1</a></li>
<li class="chapter" data-level="7.2" data-path="pubmon.html"><a href="pubmon.html#phd-year-2"><i class="fa fa-check"></i><b>7.2</b> 2021-2022 PhD year 2</a>
<ul>
<li class="chapter" data-level="7.2.1" data-path="pubmon.html"><a href="pubmon.html#resources"><i class="fa fa-check"></i><b>7.2.1</b> Resources</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">ReadingNotes: Have you read today’s papers?</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="technology-related" class="section level1" number="3">
<h1><span class="header-section-number">Chapter 3</span> Technology Related</h1>
<div id="scrna-seq-modelling" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> scRNA-seq modelling</h2>
<div id="nb-models-for-scrna-seq" class="section level3" number="3.1.1">
<h3><span class="header-section-number">3.1.1</span> NB models for scRNA-seq</h3>
<p>KEY WORDS: <strong>scRNA-seq counts</strong></p>
<p>DEseq2, scTransform, BASICS</p>
<div id="deseq2" class="section level4" number="3.1.1.1">
<h4><span class="header-section-number">3.1.1.1</span> DEseq2</h4>
<p><strong><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0550-8">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</a><span class="citation">(Love, Huber, and Anders <a href="#ref-love2014moderated" role="doc-biblioref">2014</a>)</span></strong></p>
<div class="figure">
<img src="figs/RNAseqCounts/DEseq2.png" alt="" />
<p class="caption">DEseq2</p>
</div>
<div class="figure">
<img src="figs/RNAseqCounts/DEseq2_Fig1.png" alt="" />
<p class="caption">DEseq2-fig1</p>
</div>
<p><strong>Shrinkage estimation of dispersion.</strong> Plot of dispersion estimates over the average expression strength (A) for the Bottomly et al. [16] dataset with six samples across two groups and (B) for five samples from the Pickrell et al. [17] dataset, fitting only an intercept term. First, gene-wise MLEs are obtained using only the respective genes data (black dots). Then, a curve (red) is fit to the MLEs to capture the overall trend of dispersion-mean dependence. This fit is used as a prior mean for a second estimation round, which results in the final MAP estimates of dispersion (arrow heads). This can be understood as a shrinkage (along the blue arrows) of the noisy gene-wise estimates toward the consensus represented by the red line. The black points circled in blue are detected as dispersion outliers and not shrunk toward the prior (shrinkage would follow the dotted line). For clarity, only a subset of genes is shown, which is enriched for dispersion outliers. Additional file 1: Figure S1 displays the same data but with dispersions of all genes shown. MAP, maximum a posteriori; MLE, maximum-likelihood estimate.</p>
</div>
<div id="sctransform" class="section level4" number="3.1.1.2">
<h4><span class="header-section-number">3.1.1.2</span> scTransform</h4>
<p><code>Our procedure is broadly applicable for any UMI-based scRNA-seq dataset and is freely available to users through the open-source R package sctransform (github.com/ChristophH/sctransform), with a direct interface to our single-cell toolkit Seurat.</code></p>
<p><code>**Pearson residuals** from "regularized negative binomial regression," help remove the influence of technical characteristics from downstream analyses while preserving biological heterogeneity</code></p>
<p><code>unconstrained negative binomial model may overfit scRNA-seq data, and overcome this by pooling information across genes with similar abundances to obtain stable parameter estimates.</code></p>
<p><strong><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1874-1">Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression</a><span class="citation">(Hafemeister and Satija <a href="#ref-hafemeister2019normalization" role="doc-biblioref">2019</a>)</span></strong></p>
<p><a href="https://github.com/ChristophH/sctransform/" class="uri">https://github.com/ChristophH/sctransform/</a></p>
<div class="figure">
<img src="figs/RNAseqCounts/sctransform.png" alt="" />
<p class="caption">SCTransform</p>
</div>
<p>We propose that the Pearson residuals from regularized negative binomial regression, where cellular sequencing depth is utilized as a covariate in a generalized linear model, successfully remove the influence of technical characteristics from downstream analyses while preserving biological heterogeneity.</p>
<ul>
<li>UMI-based scRNA-seq dataset</li>
</ul>
<div class="figure">
<img src="figs/RNAseqCounts/sctransform-intro.png" alt="" />
<p class="caption">SCTransform-intro</p>
</div>
<p><code>observed **sequencing depth** (number of genes or molecules detected per cell) can vary significantly between cells, with variation in molecular counts potentially spanning an order of magnitude, even within the same cell type</code></p>
<p><code>while the now widespread use of unique molecular identifiers (UMI) in scRNA-seq removes technical variation associated with PCR, differences in cell lysis, reverse transcription efficiency, and stochastic molecular sampling during sequencing also contribute significantly, necessitating technical correction</code></p>
<p><img src="figs/RNAseqCounts/background.png" alt="review" />
<img src="figs/RNAseqCounts/sctrans_intro.png" alt="SCTransform-intro" />
<strong>Results</strong></p>
<ul>
<li><p>A single scaling factor does not effectively normalize both lowly and highly expressed genes</p></li>
<li><p>Modeling single-cell data with a negative binomial distribution leads to overfitting</p></li>
<li><p>Applying regularized negative binomial regression for single-cell normalization</p></li>
<li><p>Pearson residuals effectively normalize technical differences, while retaining biological variation</p></li>
<li><p>Downstream analytical tasks are not biased by sequencing depth</p></li>
</ul>
<div class="figure">
<img src="figs/RNAseqCounts/sctransform-sequencing%20depth.png" alt="" />
<p class="caption">.</p>
</div>
<p><img src="figs/RNAseqCounts/sctransform-fig1.png" alt="." />
<img src="figs/RNAseqCounts/sctransform-figS1.png" alt="." /></p>
<p><strong>Methods</strong></p>
<ul>
<li>Regularized negative binomial regression</li>
</ul>
</div>
<div id="basics" class="section level4" number="3.1.1.3">
<h4><span class="header-section-number">3.1.1.3</span> BASiCS</h4>
<p><strong><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004333">BASiCS: Bayesian Analysis of Single-Cell Sequencing Data</a><span class="citation">(Vallejos, Marioni, and Richardson <a href="#ref-vallejos2015basics" role="doc-biblioref">2015</a>)</span></strong></p>
<p><img src="figs/RNAseqCounts/BASICS.png" alt="BASICS Abstract" />
<img src="figs/RNAseqCounts/BASICS2.png" alt="BASICS Author summary" />
<strong>INTRODUCTION</strong></p>
<ul>
<li>Cell_SPECIFIC NORMALIZATION</li>
</ul>
<p>For instance, in Fig 1(a), each gene has the same expression rate in both cells, yet the expression counts in the first cell will be roughly twice as much as those from the second cell. In the same spirit, if different sequencing depths (the number of times a single nucleotide is read during the sequencing) are applied to these cells, the scale of expression counts will also be affected. Thus, normalisation is a crucial issue in this context.</p>
<ul>
<li>gene specific variation</li>
</ul>
<p>Another fundamental problem for interpreting single-cell sequencing is the presence of high levels of unexplained technical noise (unrelated to sequencing depth and other amplification biases) [5]. This creates new challenges for identifying genes that show genuine biological cell-to-cell heterogeneity-beyond that induced by technical variation-and motivates the systematic inclusion of spike-in genes in single-cell experiments</p>
<ul>
<li>UMI</li>
</ul>
<p>Recently, the introduction of Unique Molecular Identifiers (UMI) attached to each cDNA molecule during reverse transcription has substantially reduced the levels of unexplained technical noise and eliminated the effect of sequencing depth changes and other amplification biases in single-cell experiments.</p>
<p>Nevertheless, our analysis of a mouse Embryonic Stem Cells (ESC) suggests that unexplained technical variability can not be completely removed by using UMIs (see Results section) and that an accurate quantification of technical variability still remains important.</p>
<p><strong>HIGHLIGHT</strong></p>
<p>In BASiCS (Bayesian Analysis of Single-Cell Sequencing data), a joint model of biological and spike-in genes is formulated to simultaneously quantify unexplained technical noise and cell-to-cell biological heterogeneity using the complete set of data, borrowing information between both sets of genes (spike-in and biological) through common parameters in a hierarchical structure. Additionally, BASiCS incorporates an automated normalisation method, where normalising constants are treated as model parameters.</p>
<p><img src="figs/RNAseqCounts/pcbi.1004333.g001.png" alt="Graphical representation of gene expression in two cells from a homogeneous population but with different total mRNA content." />
<img src="figs/RNAseqCounts/pcbi.1004333.g002.png" alt="Graphical representation of the hierarchical model implemented in BASiCS." />
<img src="figs/RNAseqCounts/BASiCS-FIG2.png" alt="Graphical representation of the hierarchical model implemented in BASiCS." />
#### edgeR</p>
<p><strong><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2796818/">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</a><span class="citation">(Robinson, McCarthy, and Smyth <a href="#ref-robinson2010edger" role="doc-biblioref">2010</a>)</span></strong></p>
<p><strong><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3378882/">Differential expression analysis of multifactor RNA-Seq experiments with respect to biological variation</a><span class="citation">(McCarthy, Chen, and Smyth <a href="#ref-mccarthy2012differential" role="doc-biblioref">2012</a>)</span></strong></p>
</div>
<div id="sanity" class="section level4" number="3.1.1.4">
<h4><span class="header-section-number">3.1.1.4</span> Sanity</h4>
<p><strong><a href="https://www.nature.com/articles/s41587-021-00875-x">Bayesian inference of gene expression states from single-cell RNA-seq data</a><span class="citation">(Breda, Zavolan, and Nimwegen <a href="#ref-breda2021bayesian" role="doc-biblioref">2021</a>)</span></strong></p>
</div>
</div>
<div id="batch-effect-correction-methods-for-scrna-seq" class="section level3" number="3.1.2">
<h3><span class="header-section-number">3.1.2</span> Batch-effect correction methods for scRNA-seq</h3>
<p><strong><a href="https://link.springer.com/article/10.1186/s13059-019-1850-9">A benchmark of batch-effect correction methods for single-cell RNA sequencing data</a><span class="citation">(Tran et al. <a href="#ref-tran2020benchmark" role="doc-biblioref">2020</a>)</span></strong></p>
<ul>
<li>Abstract
<strong>Background</strong>: Large-scale single-cell transcriptomic datasets generated using different technologies contain batchspecific
systematic variations that present a challenge to batch-effect removal and data integration. With continued
growth expected in scRNA-seq data, achieving effective batch integration with available computational resources is
crucial. Here, we perform an in-depth benchmark study on available batch correction methods to determine the
most suitable method for batch-effect removal.</li>
</ul>
<p><strong>Results</strong>: We compare 14 methods in terms of computational runtime, the ability to handle large datasets, and
batch-effect correction efficacy while preserving cell type purity. Five scenarios are designed for the study: identical
cell types with different technologies, non-identical cell types, multiple batches, big data, and simulated data.
Performance is evaluated using four benchmarking metrics including kBET, LISI, ASW, and ARI. We also investigate
the use of batch-corrected data to study differential gene expression.</p>
<p><strong>Conclusion</strong>: Based on our results, Harmony, LIGER, and Seurat 3 are the recommended methods for batch
integration. Due to its significantly shorter runtime, Harmony is recommended as the first method to try, with the
other methods as viable alternatives.</p>
<p><strong>Keywords</strong>: Single-cell RNA-seq, Batch correction, Batch effect, Integration, Differential gene expression</p>
</div>
</div>
<div id="hi-seq" class="section level2" number="3.2">
<h2><span class="header-section-number">3.2</span> Hi-Seq</h2>
<p><strong><a href="https://www.nature.com/articles/s41587-021-00981-w">Enhanced detection of minimal residual disease by targeted sequencing of phased variants in circulating tumor DNA</a><span class="citation">(Kurtz et al. <a href="#ref-Kurtz2021" role="doc-biblioref">2021</a>)</span></strong></p>
<p>KEY WORDS: <strong>phased variants</strong></p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references hanging-indent">
<div id="ref-breda2021bayesian">
<p>Breda, Jérémie, Mihaela Zavolan, and Erik van Nimwegen. 2021. “Bayesian Inference of Gene Expression States from Single-Cell Rna-Seq Data.” <em>Nature Biotechnology</em>, 1–9. <a href="https://www.nature.com/articles/s41587-021-00875-x">https://www.nature.com/articles/s41587-021-00875-x</a>.</p>
</div>
<div id="ref-hafemeister2019normalization">
<p>Hafemeister, Christoph, and Rahul Satija. 2019. “Normalization and Variance Stabilization of Single-Cell Rna-Seq Data Using Regularized Negative Binomial Regression.” <em>Genome Biology</em> 20 (1): 1–15.</p>
</div>
<div id="ref-Kurtz2021">
<p>Kurtz, David M., Joanne Soo, Lyron Co Ting Keh, Stefan Alig, Jacob J. Chabon, Brian J. Sworder, Andre Schultz, et al. 2021. “Enhanced Detection of Minimal Residual Disease by Targeted Sequencing of Phased Variants in Circulating Tumor Dna.” <em>Nature Biotechnology</em>, July. <a href="https://doi.org/10.1038/s41587-021-00981-w">https://doi.org/10.1038/s41587-021-00981-w</a>.</p>
</div>
<div id="ref-love2014moderated">
<p>Love, Michael I, Wolfgang Huber, and Simon Anders. 2014. “Moderated Estimation of Fold Change and Dispersion for Rna-Seq Data with Deseq2.” <em>Genome Biology</em> 15 (12): 1–21.</p>
</div>
<div id="ref-mccarthy2012differential">
<p>McCarthy, Davis J, Yunshun Chen, and Gordon K Smyth. 2012. “Differential Expression Analysis of Multifactor Rna-Seq Experiments with Respect to Biological Variation.” <em>Nucleic Acids Research</em> 40 (10): 4288–97.</p>
</div>
<div id="ref-robinson2010edger">
<p>Robinson, Mark D, Davis J McCarthy, and Gordon K Smyth. 2010. “EdgeR: A Bioconductor Package for Differential Expression Analysis of Digital Gene Expression Data.” <em>Bioinformatics</em> 26 (1): 139–40.</p>
</div>
<div id="ref-tran2020benchmark">
<p>Tran, Hoa Thi Nhu, Kok Siong Ang, Marion Chevrier, Xiaomeng Zhang, Nicole Yee Shin Lee, Michelle Goh, and Jinmiao Chen. 2020. “A Benchmark of Batch-Effect Correction Methods for Single-Cell Rna Sequencing Data.” <em>Genome Biology</em> 21 (1): 1–32. <a href="https://link.springer.com/article/10.1186/s13059-019-1850-9">https://link.springer.com/article/10.1186/s13059-019-1850-9</a>.</p>
</div>
<div id="ref-vallejos2015basics">
<p>Vallejos, Catalina A, John C Marioni, and Sylvia Richardson. 2015. “BASiCS: Bayesian Analysis of Single-Cell Sequencing Data.” <em>PLoS Computational Biology</em> 11 (6): e1004333.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="sgcell.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="statmethods.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["ReadingNotes.pdf", "ReadingNotes.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
